SOURCE=>BASE: Node(<org.jdbi.jdbi3.Handle$1: org.jdbi.jdbi3.ResultSetRow computeNext()>,$stack10 = new org.jdbi.jdbi3.JDBIException,55,SourceNode, path: (<org.jdbi.jdbi3.Handle$1: org.jdbi.jdbi3.ResultSetRow computeNext()>, specialinvoke $stack10.<org.jdbi.jdbi3.JDBIException: void <init>(java.lang.Throwable)>(e), 55)) => Node(<org.jdbi.jdbi3.JDBIException: void <init>(java.lang.Throwable)>,this := @this: org.jdbi.jdbi3.JDBIException,-1,SimpleNode, path: (<org.jdbi.jdbi3.Handle$1: org.jdbi.jdbi3.ResultSetRow computeNext()>, specialinvoke $stack10.<org.jdbi.jdbi3.JDBIException: void <init>(java.lang.Throwable)>(e), 55)) => Node(<java.lang.RuntimeException: void <init>(java.lang.Throwable)>,l0 := @this: java.lang.RuntimeException,-1,SimpleNode, path: (<org.jdbi.jdbi3.Handle$1: org.jdbi.jdbi3.ResultSetRow computeNext()>, specialinvoke $stack10.<org.jdbi.jdbi3.JDBIException: void <init>(java.lang.Throwable)>(e), 55) => (<org.jdbi.jdbi3.JDBIException: void <init>(java.lang.Throwable)>, specialinvoke this.<java.lang.RuntimeException: void <init>(java.lang.Throwable)>(cause), 7)) => Node(<java.lang.RuntimeException: void <init>(java.lang.Throwable)>,specialinvoke l0.<java.lang.Exception: void <init>(java.lang.Throwable)>(l1),96,SinkNode, path: (<org.jdbi.jdbi3.Handle$1: org.jdbi.jdbi3.ResultSetRow computeNext()>, specialinvoke $stack10.<org.jdbi.jdbi3.JDBIException: void <init>(java.lang.Throwable)>(e), 55) => (<org.jdbi.jdbi3.JDBIException: void <init>(java.lang.Throwable)>, specialinvoke this.<java.lang.RuntimeException: void <init>(java.lang.Throwable)>(cause), 7) => (<java.lang.RuntimeException: void <init>(java.lang.Throwable)>, specialinvoke l0.<java.lang.Exception: void <init>(java.lang.Throwable)>(l1), 96))
SINK=>BASE: Node(<org.jdbi.jdbi3.Handle: java.util.stream.Stream query(java.lang.String)>,$stack11 := @caughtexception,60,SourceNode, path: (<org.jdbi.jdbi3.Handle: java.util.stream.Stream query(java.lang.String)>, stmt#2 = $stack11, 62)) => Node(<org.jdbi.jdbi3.Handle: java.util.stream.Stream query(java.lang.String)>,stmt#2 = $stack11,62,SinkNode, path: (<org.jdbi.jdbi3.Handle: java.util.stream.Stream query(java.lang.String)>, stmt#2 = $stack11, 62)) => Node(<org.jdbi.jdbi3.JDBIException: void <init>(java.lang.Throwable)>,cause := @parameter0: java.lang.Throwable,-1,SimpleNode, path: (<org.jdbi.jdbi3.Handle$1: org.jdbi.jdbi3.ResultSetRow computeNext()>, specialinvoke $stack10.<org.jdbi.jdbi3.JDBIException: void <init>(java.lang.Throwable)>(e), 55)) => Node(<java.lang.RuntimeException: void <init>(java.lang.Throwable)>,l1 := @parameter0: java.lang.Throwable,-1,SimpleNode, path: (<org.jdbi.jdbi3.Handle$1: org.jdbi.jdbi3.ResultSetRow computeNext()>, specialinvoke $stack10.<org.jdbi.jdbi3.JDBIException: void <init>(java.lang.Throwable)>(e), 55) => (<org.jdbi.jdbi3.JDBIException: void <init>(java.lang.Throwable)>, specialinvoke this.<java.lang.RuntimeException: void <init>(java.lang.Throwable)>(cause), 7)) => Node(<java.lang.RuntimeException: void <init>(java.lang.Throwable)>,specialinvoke l0.<java.lang.Exception: void <init>(java.lang.Throwable)>(l1),96,SinkNode, path: (<org.jdbi.jdbi3.Handle$1: org.jdbi.jdbi3.ResultSetRow computeNext()>, specialinvoke $stack10.<org.jdbi.jdbi3.JDBIException: void <init>(java.lang.Throwable)>(e), 55) => (<org.jdbi.jdbi3.JDBIException: void <init>(java.lang.Throwable)>, specialinvoke this.<java.lang.RuntimeException: void <init>(java.lang.Throwable)>(cause), 7) => (<java.lang.RuntimeException: void <init>(java.lang.Throwable)>, specialinvoke l0.<java.lang.Exception: void <init>(java.lang.Throwable)>(l1), 96))

SOURCE=>BASE: Node(<org.jdbi.jdbi3.Handle: java.util.stream.Stream query(java.lang.String)>,$stack11 := @caughtexception,60,SourceNode, path: (<org.jdbi.jdbi3.Handle: java.util.stream.Stream query(java.lang.String)>, stmt#2 = $stack11, 62)) => Node(<org.jdbi.jdbi3.Handle: java.util.stream.Stream query(java.lang.String)>,stmt#2 = $stack11,62,SinkNode, path: (<org.jdbi.jdbi3.Handle: java.util.stream.Stream query(java.lang.String)>, stmt#2 = $stack11, 62)) => Node(<org.jdbi.jdbi3.JDBIException: void <init>(java.lang.Throwable)>,cause := @parameter0: java.lang.Throwable,-1,SimpleNode, path: (<org.jdbi.jdbi3.Handle$1: org.jdbi.jdbi3.ResultSetRow computeNext()>, specialinvoke $stack10.<org.jdbi.jdbi3.JDBIException: void <init>(java.lang.Throwable)>(e), 55)) => Node(<org.jdbi.jdbi3.JDBIException: void <init>(java.lang.Throwable)>,specialinvoke this.<java.lang.RuntimeException: void <init>(java.lang.Throwable)>(cause),7,SinkNode, path: (<org.jdbi.jdbi3.Handle$1: org.jdbi.jdbi3.ResultSetRow computeNext()>, specialinvoke $stack10.<org.jdbi.jdbi3.JDBIException: void <init>(java.lang.Throwable)>(e), 55) => (<org.jdbi.jdbi3.JDBIException: void <init>(java.lang.Throwable)>, specialinvoke this.<java.lang.RuntimeException: void <init>(java.lang.Throwable)>(cause), 7))
SINK=>BASE: Node(<org.jdbi.jdbi3.Handle$1: org.jdbi.jdbi3.ResultSetRow computeNext()>,$stack9 := @caughtexception,45,SourceNode, path: (<org.jdbi.jdbi3.Handle$1: org.jdbi.jdbi3.ResultSetRow computeNext()>, e = $stack9, 53)) => Node(<org.jdbi.jdbi3.Handle$1: org.jdbi.jdbi3.ResultSetRow computeNext()>,e = $stack9,53,SourceNode, path: (<org.jdbi.jdbi3.Handle$1: org.jdbi.jdbi3.ResultSetRow computeNext()>, e = $stack9, 53)) => Node(<org.jdbi.jdbi3.JDBIException: void <init>(java.lang.Throwable)>,cause := @parameter0: java.lang.Throwable,-1,SimpleNode, path: (<org.jdbi.jdbi3.Handle$1: org.jdbi.jdbi3.ResultSetRow computeNext()>, specialinvoke $stack10.<org.jdbi.jdbi3.JDBIException: void <init>(java.lang.Throwable)>(e), 55)) => Node(<org.jdbi.jdbi3.JDBIException: void <init>(java.lang.Throwable)>,specialinvoke this.<java.lang.RuntimeException: void <init>(java.lang.Throwable)>(cause),7,SinkNode, path: (<org.jdbi.jdbi3.Handle$1: org.jdbi.jdbi3.ResultSetRow computeNext()>, specialinvoke $stack10.<org.jdbi.jdbi3.JDBIException: void <init>(java.lang.Throwable)>(e), 55) => (<org.jdbi.jdbi3.JDBIException: void <init>(java.lang.Throwable)>, specialinvoke this.<java.lang.RuntimeException: void <init>(java.lang.Throwable)>(cause), 7))

SOURCE=>BASE: Node(<org.jdbi.jdbi3.Handle$1: org.jdbi.jdbi3.ResultSetRow computeNext()>,$stack10 = new org.jdbi.jdbi3.JDBIException,55,SourceNode, path: (<org.jdbi.jdbi3.Handle$1: org.jdbi.jdbi3.ResultSetRow computeNext()>, specialinvoke $stack10.<org.jdbi.jdbi3.JDBIException: void <init>(java.lang.Throwable)>(e), 55)) => Node(<org.jdbi.jdbi3.JDBIException: void <init>(java.lang.Throwable)>,this := @this: org.jdbi.jdbi3.JDBIException,-1,SimpleNode, path: (<org.jdbi.jdbi3.Handle$1: org.jdbi.jdbi3.ResultSetRow computeNext()>, specialinvoke $stack10.<org.jdbi.jdbi3.JDBIException: void <init>(java.lang.Throwable)>(e), 55)) => Node(<java.lang.RuntimeException: void <init>(java.lang.Throwable)>,l0 := @this: java.lang.RuntimeException,-1,SimpleNode, path: (<org.jdbi.jdbi3.Handle$1: org.jdbi.jdbi3.ResultSetRow computeNext()>, specialinvoke $stack10.<org.jdbi.jdbi3.JDBIException: void <init>(java.lang.Throwable)>(e), 55) => (<org.jdbi.jdbi3.JDBIException: void <init>(java.lang.Throwable)>, specialinvoke this.<java.lang.RuntimeException: void <init>(java.lang.Throwable)>(cause), 7)) => Node(<java.lang.Exception: void <init>(java.lang.Throwable)>,l0 := @this: java.lang.Exception,-1,SimpleNode, path: (<org.jdbi.jdbi3.Handle$1: org.jdbi.jdbi3.ResultSetRow computeNext()>, specialinvoke $stack10.<org.jdbi.jdbi3.JDBIException: void <init>(java.lang.Throwable)>(e), 55) => (<org.jdbi.jdbi3.JDBIException: void <init>(java.lang.Throwable)>, specialinvoke this.<java.lang.RuntimeException: void <init>(java.lang.Throwable)>(cause), 7) => (<java.lang.RuntimeException: void <init>(java.lang.Throwable)>, specialinvoke l0.<java.lang.Exception: void <init>(java.lang.Throwable)>(l1), 96)) => Node(<java.lang.Throwable: void <init>(java.lang.Throwable)>,l0 := @this: java.lang.Throwable,-1,SimpleNode, path: (<org.jdbi.jdbi3.Handle$1: org.jdbi.jdbi3.ResultSetRow computeNext()>, specialinvoke $stack10.<org.jdbi.jdbi3.JDBIException: void <init>(java.lang.Throwable)>(e), 55) => (<org.jdbi.jdbi3.JDBIException: void <init>(java.lang.Throwable)>, specialinvoke this.<java.lang.RuntimeException: void <init>(java.lang.Throwable)>(cause), 7) => (<java.lang.RuntimeException: void <init>(java.lang.Throwable)>, specialinvoke l0.<java.lang.Exception: void <init>(java.lang.Throwable)>(l1), 96) => (<java.lang.Exception: void <init>(java.lang.Throwable)>, specialinvoke l0.<java.lang.Throwable: void <init>(java.lang.Throwable)>(l1), 102)) => Node(<java.lang.Throwable: void <init>(java.lang.Throwable)>,l0.<java.lang.Throwable: java.lang.Throwable cause> = l1,312,SinkNode, path: (<org.jdbi.jdbi3.Handle$1: org.jdbi.jdbi3.ResultSetRow computeNext()>, specialinvoke $stack10.<org.jdbi.jdbi3.JDBIException: void <init>(java.lang.Throwable)>(e), 55) => (<org.jdbi.jdbi3.JDBIException: void <init>(java.lang.Throwable)>, specialinvoke this.<java.lang.RuntimeException: void <init>(java.lang.Throwable)>(cause), 7) => (<java.lang.RuntimeException: void <init>(java.lang.Throwable)>, specialinvoke l0.<java.lang.Exception: void <init>(java.lang.Throwable)>(l1), 96) => (<java.lang.Exception: void <init>(java.lang.Throwable)>, specialinvoke l0.<java.lang.Throwable: void <init>(java.lang.Throwable)>(l1), 102) => (<java.lang.Throwable: void <init>(java.lang.Throwable)>, l0.<java.lang.Throwable: java.lang.Throwable cause> = l1, 312))
SINK=>BASE: Node(<org.jdbi.jdbi3.Handle: java.util.stream.Stream query(java.lang.String)>,$stack11 := @caughtexception,60,SourceNode, path: (<org.jdbi.jdbi3.Handle: java.util.stream.Stream query(java.lang.String)>, stmt#2 = $stack11, 62)) => Node(<org.jdbi.jdbi3.Handle: java.util.stream.Stream query(java.lang.String)>,stmt#2 = $stack11,62,SinkNode, path: (<org.jdbi.jdbi3.Handle: java.util.stream.Stream query(java.lang.String)>, stmt#2 = $stack11, 62)) => Node(<org.jdbi.jdbi3.JDBIException: void <init>(java.lang.Throwable)>,cause := @parameter0: java.lang.Throwable,-1,SimpleNode, path: (<org.jdbi.jdbi3.Handle$1: org.jdbi.jdbi3.ResultSetRow computeNext()>, specialinvoke $stack10.<org.jdbi.jdbi3.JDBIException: void <init>(java.lang.Throwable)>(e), 55)) => Node(<java.lang.RuntimeException: void <init>(java.lang.Throwable)>,l1 := @parameter0: java.lang.Throwable,-1,SimpleNode, path: (<org.jdbi.jdbi3.Handle$1: org.jdbi.jdbi3.ResultSetRow computeNext()>, specialinvoke $stack10.<org.jdbi.jdbi3.JDBIException: void <init>(java.lang.Throwable)>(e), 55) => (<org.jdbi.jdbi3.JDBIException: void <init>(java.lang.Throwable)>, specialinvoke this.<java.lang.RuntimeException: void <init>(java.lang.Throwable)>(cause), 7)) => Node(<java.lang.Exception: void <init>(java.lang.Throwable)>,l1 := @parameter0: java.lang.Throwable,-1,SimpleNode, path: (<org.jdbi.jdbi3.Handle$1: org.jdbi.jdbi3.ResultSetRow computeNext()>, specialinvoke $stack10.<org.jdbi.jdbi3.JDBIException: void <init>(java.lang.Throwable)>(e), 55) => (<org.jdbi.jdbi3.JDBIException: void <init>(java.lang.Throwable)>, specialinvoke this.<java.lang.RuntimeException: void <init>(java.lang.Throwable)>(cause), 7) => (<java.lang.RuntimeException: void <init>(java.lang.Throwable)>, specialinvoke l0.<java.lang.Exception: void <init>(java.lang.Throwable)>(l1), 96)) => Node(<java.lang.Throwable: void <init>(java.lang.Throwable)>,l1 := @parameter0: java.lang.Throwable,-1,SimpleNode, path: (<org.jdbi.jdbi3.Handle$1: org.jdbi.jdbi3.ResultSetRow computeNext()>, specialinvoke $stack10.<org.jdbi.jdbi3.JDBIException: void <init>(java.lang.Throwable)>(e), 55) => (<org.jdbi.jdbi3.JDBIException: void <init>(java.lang.Throwable)>, specialinvoke this.<java.lang.RuntimeException: void <init>(java.lang.Throwable)>(cause), 7) => (<java.lang.RuntimeException: void <init>(java.lang.Throwable)>, specialinvoke l0.<java.lang.Exception: void <init>(java.lang.Throwable)>(l1), 96) => (<java.lang.Exception: void <init>(java.lang.Throwable)>, specialinvoke l0.<java.lang.Throwable: void <init>(java.lang.Throwable)>(l1), 102)) => Node(<java.lang.Throwable: void <init>(java.lang.Throwable)>,l0.<java.lang.Throwable: java.lang.Throwable cause> = l1,312,SinkNode, path: (<org.jdbi.jdbi3.Handle$1: org.jdbi.jdbi3.ResultSetRow computeNext()>, specialinvoke $stack10.<org.jdbi.jdbi3.JDBIException: void <init>(java.lang.Throwable)>(e), 55) => (<org.jdbi.jdbi3.JDBIException: void <init>(java.lang.Throwable)>, specialinvoke this.<java.lang.RuntimeException: void <init>(java.lang.Throwable)>(cause), 7) => (<java.lang.RuntimeException: void <init>(java.lang.Throwable)>, specialinvoke l0.<java.lang.Exception: void <init>(java.lang.Throwable)>(l1), 96) => (<java.lang.Exception: void <init>(java.lang.Throwable)>, specialinvoke l0.<java.lang.Throwable: void <init>(java.lang.Throwable)>(l1), 102) => (<java.lang.Throwable: void <init>(java.lang.Throwable)>, l0.<java.lang.Throwable: java.lang.Throwable cause> = l1, 312))

SOURCE=>BASE: Node(<org.jdbi.jdbi3.Handle: java.util.stream.Stream query(java.lang.String)>,$stack11 := @caughtexception,60,SourceNode, path: (<org.jdbi.jdbi3.Handle: java.util.stream.Stream query(java.lang.String)>, stmt#2 = $stack11, 62)) => Node(<org.jdbi.jdbi3.Handle: java.util.stream.Stream query(java.lang.String)>,stmt#2 = $stack11,62,SinkNode, path: (<org.jdbi.jdbi3.Handle: java.util.stream.Stream query(java.lang.String)>, stmt#2 = $stack11, 62)) => Node(<org.jdbi.jdbi3.Handle: java.util.stream.Stream query(java.lang.String)>,specialinvoke tmp$1952194564.<org.jdbi.jdbi3.JDBIException: void <init>(java.lang.Throwable)>(stmt#2),64,SinkNode, path: (<org.jdbi.jdbi3.Handle: java.util.stream.Stream query(java.lang.String)>, specialinvoke tmp$1952194564.<org.jdbi.jdbi3.JDBIException: void <init>(java.lang.Throwable)>(stmt#2), 64))
SINK=>BASE: Node(<org.jdbi.jdbi3.Handle: java.util.stream.Stream query(java.lang.String)>,$stack11 := @caughtexception,60,SourceNode, path: (<org.jdbi.jdbi3.Handle: java.util.stream.Stream query(java.lang.String)>, stmt#2 = $stack11, 62)) => Node(<org.jdbi.jdbi3.Handle: java.util.stream.Stream query(java.lang.String)>,stmt#2 = $stack11,62,SinkNode, path: (<org.jdbi.jdbi3.Handle: java.util.stream.Stream query(java.lang.String)>, stmt#2 = $stack11, 62)) => Node(<org.jdbi.jdbi3.Handle: java.util.stream.Stream query(java.lang.String)>,specialinvoke tmp$1040493509.<org.jdbi.jdbi3.JDBIException: void <init>(java.lang.Throwable)>(stmt#2),64,SinkNode, path: (<org.jdbi.jdbi3.Handle: java.util.stream.Stream query(java.lang.String)>, specialinvoke tmp$1040493509.<org.jdbi.jdbi3.JDBIException: void <init>(java.lang.Throwable)>(stmt#2), 64))

SOURCE=>BASE: Node(<org.jdbi.jdbi3.Handle: java.util.stream.Stream query(java.lang.String)>,$stack11 := @caughtexception,60,SourceNode, path: (<org.jdbi.jdbi3.Handle: java.util.stream.Stream query(java.lang.String)>, stmt#2 = $stack11, 62)) => Node(<org.jdbi.jdbi3.Handle: java.util.stream.Stream query(java.lang.String)>,stmt#2 = $stack11,62,SinkNode, path: (<org.jdbi.jdbi3.Handle: java.util.stream.Stream query(java.lang.String)>, stmt#2 = $stack11, 62)) => Node(<org.jdbi.jdbi3.JDBIException: void <init>(java.lang.Throwable)>,cause := @parameter0: java.lang.Throwable,-1,SimpleNode, path: (<org.jdbi.jdbi3.Handle$1: org.jdbi.jdbi3.ResultSetRow computeNext()>, specialinvoke $stack10.<org.jdbi.jdbi3.JDBIException: void <init>(java.lang.Throwable)>(e), 55)) => Node(<java.lang.RuntimeException: void <init>(java.lang.Throwable)>,l1 := @parameter0: java.lang.Throwable,-1,SimpleNode, path: (<org.jdbi.jdbi3.Handle$1: org.jdbi.jdbi3.ResultSetRow computeNext()>, specialinvoke $stack10.<org.jdbi.jdbi3.JDBIException: void <init>(java.lang.Throwable)>(e), 55) => (<org.jdbi.jdbi3.JDBIException: void <init>(java.lang.Throwable)>, specialinvoke this.<java.lang.RuntimeException: void <init>(java.lang.Throwable)>(cause), 7)) => Node(<java.lang.Exception: void <init>(java.lang.Throwable)>,l1 := @parameter0: java.lang.Throwable,-1,SimpleNode, path: (<org.jdbi.jdbi3.Handle$1: org.jdbi.jdbi3.ResultSetRow computeNext()>, specialinvoke $stack10.<org.jdbi.jdbi3.JDBIException: void <init>(java.lang.Throwable)>(e), 55) => (<org.jdbi.jdbi3.JDBIException: void <init>(java.lang.Throwable)>, specialinvoke this.<java.lang.RuntimeException: void <init>(java.lang.Throwable)>(cause), 7) => (<java.lang.RuntimeException: void <init>(java.lang.Throwable)>, specialinvoke l0.<java.lang.Exception: void <init>(java.lang.Throwable)>(l1), 96)) => Node(<java.lang.Exception: void <init>(java.lang.Throwable)>,specialinvoke l0.<java.lang.Throwable: void <init>(java.lang.Throwable)>(l1),102,SinkNode, path: (<org.jdbi.jdbi3.Handle$1: org.jdbi.jdbi3.ResultSetRow computeNext()>, specialinvoke $stack10.<org.jdbi.jdbi3.JDBIException: void <init>(java.lang.Throwable)>(e), 55) => (<org.jdbi.jdbi3.JDBIException: void <init>(java.lang.Throwable)>, specialinvoke this.<java.lang.RuntimeException: void <init>(java.lang.Throwable)>(cause), 7) => (<java.lang.RuntimeException: void <init>(java.lang.Throwable)>, specialinvoke l0.<java.lang.Exception: void <init>(java.lang.Throwable)>(l1), 96) => (<java.lang.Exception: void <init>(java.lang.Throwable)>, specialinvoke l0.<java.lang.Throwable: void <init>(java.lang.Throwable)>(l1), 102))
SINK=>BASE: Node(<org.jdbi.jdbi3.Handle$1: org.jdbi.jdbi3.ResultSetRow computeNext()>,e = $stack9,53,SourceNode, path: (<org.jdbi.jdbi3.Handle$1: org.jdbi.jdbi3.ResultSetRow computeNext()>, e = $stack9, 53)) => Node(<org.jdbi.jdbi3.JDBIException: void <init>(java.lang.Throwable)>,cause := @parameter0: java.lang.Throwable,-1,SimpleNode, path: (<org.jdbi.jdbi3.Handle$1: org.jdbi.jdbi3.ResultSetRow computeNext()>, specialinvoke $stack10.<org.jdbi.jdbi3.JDBIException: void <init>(java.lang.Throwable)>(e), 55)) => Node(<java.lang.RuntimeException: void <init>(java.lang.Throwable)>,l1 := @parameter0: java.lang.Throwable,-1,SimpleNode, path: (<org.jdbi.jdbi3.Handle$1: org.jdbi.jdbi3.ResultSetRow computeNext()>, specialinvoke $stack10.<org.jdbi.jdbi3.JDBIException: void <init>(java.lang.Throwable)>(e), 55) => (<org.jdbi.jdbi3.JDBIException: void <init>(java.lang.Throwable)>, specialinvoke this.<java.lang.RuntimeException: void <init>(java.lang.Throwable)>(cause), 7)) => Node(<java.lang.Exception: void <init>(java.lang.Throwable)>,l1 := @parameter0: java.lang.Throwable,-1,SimpleNode, path: (<org.jdbi.jdbi3.Handle$1: org.jdbi.jdbi3.ResultSetRow computeNext()>, specialinvoke $stack10.<org.jdbi.jdbi3.JDBIException: void <init>(java.lang.Throwable)>(e), 55) => (<org.jdbi.jdbi3.JDBIException: void <init>(java.lang.Throwable)>, specialinvoke this.<java.lang.RuntimeException: void <init>(java.lang.Throwable)>(cause), 7) => (<java.lang.RuntimeException: void <init>(java.lang.Throwable)>, specialinvoke l0.<java.lang.Exception: void <init>(java.lang.Throwable)>(l1), 96)) => Node(<java.lang.Exception: void <init>(java.lang.Throwable)>,specialinvoke l0.<java.lang.Throwable: void <init>(java.lang.Throwable)>(l1),102,SinkNode, path: (<org.jdbi.jdbi3.Handle$1: org.jdbi.jdbi3.ResultSetRow computeNext()>, specialinvoke $stack10.<org.jdbi.jdbi3.JDBIException: void <init>(java.lang.Throwable)>(e), 55) => (<org.jdbi.jdbi3.JDBIException: void <init>(java.lang.Throwable)>, specialinvoke this.<java.lang.RuntimeException: void <init>(java.lang.Throwable)>(cause), 7) => (<java.lang.RuntimeException: void <init>(java.lang.Throwable)>, specialinvoke l0.<java.lang.Exception: void <init>(java.lang.Throwable)>(l1), 96) => (<java.lang.Exception: void <init>(java.lang.Throwable)>, specialinvoke l0.<java.lang.Throwable: void <init>(java.lang.Throwable)>(l1), 102))

SOURCE=>BASE: Node(<org.jdbi.jdbi3.Handle$1: org.jdbi.jdbi3.ResultSetRow computeNext()>,$stack10 = new org.jdbi.jdbi3.JDBIException,55,SourceNode, path: (<org.jdbi.jdbi3.Handle$1: org.jdbi.jdbi3.ResultSetRow computeNext()>, specialinvoke $stack10.<org.jdbi.jdbi3.JDBIException: void <init>(java.lang.Throwable)>(e), 55)) => Node(<org.jdbi.jdbi3.JDBIException: void <init>(java.lang.Throwable)>,this := @this: org.jdbi.jdbi3.JDBIException,-1,SimpleNode, path: (<org.jdbi.jdbi3.Handle$1: org.jdbi.jdbi3.ResultSetRow computeNext()>, specialinvoke $stack10.<org.jdbi.jdbi3.JDBIException: void <init>(java.lang.Throwable)>(e), 55)) => Node(<java.lang.RuntimeException: void <init>(java.lang.Throwable)>,l0 := @this: java.lang.RuntimeException,-1,SimpleNode, path: (<org.jdbi.jdbi3.Handle$1: org.jdbi.jdbi3.ResultSetRow computeNext()>, specialinvoke $stack10.<org.jdbi.jdbi3.JDBIException: void <init>(java.lang.Throwable)>(e), 55) => (<org.jdbi.jdbi3.JDBIException: void <init>(java.lang.Throwable)>, specialinvoke this.<java.lang.RuntimeException: void <init>(java.lang.Throwable)>(cause), 7)) => Node(<java.lang.RuntimeException: void <init>(java.lang.Throwable)>,specialinvoke l0.<java.lang.Exception: void <init>(java.lang.Throwable)>(l1),96,SinkNode, path: (<org.jdbi.jdbi3.Handle$1: org.jdbi.jdbi3.ResultSetRow computeNext()>, specialinvoke $stack10.<org.jdbi.jdbi3.JDBIException: void <init>(java.lang.Throwable)>(e), 55) => (<org.jdbi.jdbi3.JDBIException: void <init>(java.lang.Throwable)>, specialinvoke this.<java.lang.RuntimeException: void <init>(java.lang.Throwable)>(cause), 7) => (<java.lang.RuntimeException: void <init>(java.lang.Throwable)>, specialinvoke l0.<java.lang.Exception: void <init>(java.lang.Throwable)>(l1), 96))
SINK=>BASE: Node(<org.jdbi.jdbi3.Handle$1: org.jdbi.jdbi3.ResultSetRow computeNext()>,e = $stack9,53,SourceNode, path: (<org.jdbi.jdbi3.Handle$1: org.jdbi.jdbi3.ResultSetRow computeNext()>, e = $stack9, 53)) => Node(<org.jdbi.jdbi3.JDBIException: void <init>(java.lang.Throwable)>,cause := @parameter0: java.lang.Throwable,-1,SimpleNode, path: (<org.jdbi.jdbi3.Handle$1: org.jdbi.jdbi3.ResultSetRow computeNext()>, specialinvoke $stack10.<org.jdbi.jdbi3.JDBIException: void <init>(java.lang.Throwable)>(e), 55)) => Node(<java.lang.RuntimeException: void <init>(java.lang.Throwable)>,l1 := @parameter0: java.lang.Throwable,-1,SimpleNode, path: (<org.jdbi.jdbi3.Handle$1: org.jdbi.jdbi3.ResultSetRow computeNext()>, specialinvoke $stack10.<org.jdbi.jdbi3.JDBIException: void <init>(java.lang.Throwable)>(e), 55) => (<org.jdbi.jdbi3.JDBIException: void <init>(java.lang.Throwable)>, specialinvoke this.<java.lang.RuntimeException: void <init>(java.lang.Throwable)>(cause), 7)) => Node(<java.lang.RuntimeException: void <init>(java.lang.Throwable)>,specialinvoke l0.<java.lang.Exception: void <init>(java.lang.Throwable)>(l1),96,SinkNode, path: (<org.jdbi.jdbi3.Handle$1: org.jdbi.jdbi3.ResultSetRow computeNext()>, specialinvoke $stack10.<org.jdbi.jdbi3.JDBIException: void <init>(java.lang.Throwable)>(e), 55) => (<org.jdbi.jdbi3.JDBIException: void <init>(java.lang.Throwable)>, specialinvoke this.<java.lang.RuntimeException: void <init>(java.lang.Throwable)>(cause), 7) => (<java.lang.RuntimeException: void <init>(java.lang.Throwable)>, specialinvoke l0.<java.lang.Exception: void <init>(java.lang.Throwable)>(l1), 96))

SOURCE=>BASE: Node(<org.jdbi.jdbi3.Handle: java.util.stream.Stream query(java.lang.String)>,$stack11 := @caughtexception,60,SourceNode, path: (<org.jdbi.jdbi3.Handle: java.util.stream.Stream query(java.lang.String)>, stmt#2 = $stack11, 62)) => Node(<org.jdbi.jdbi3.Handle: java.util.stream.Stream query(java.lang.String)>,stmt#2 = $stack11,62,SinkNode, path: (<org.jdbi.jdbi3.Handle: java.util.stream.Stream query(java.lang.String)>, stmt#2 = $stack11, 62)) => Node(<org.jdbi.jdbi3.JDBIException: void <init>(java.lang.Throwable)>,cause := @parameter0: java.lang.Throwable,-1,SimpleNode, path: (<org.jdbi.jdbi3.Handle$1: org.jdbi.jdbi3.ResultSetRow computeNext()>, specialinvoke $stack10.<org.jdbi.jdbi3.JDBIException: void <init>(java.lang.Throwable)>(e), 55)) => Node(<java.lang.RuntimeException: void <init>(java.lang.Throwable)>,l1 := @parameter0: java.lang.Throwable,-1,SimpleNode, path: (<org.jdbi.jdbi3.Handle$1: org.jdbi.jdbi3.ResultSetRow computeNext()>, specialinvoke $stack10.<org.jdbi.jdbi3.JDBIException: void <init>(java.lang.Throwable)>(e), 55) => (<org.jdbi.jdbi3.JDBIException: void <init>(java.lang.Throwable)>, specialinvoke this.<java.lang.RuntimeException: void <init>(java.lang.Throwable)>(cause), 7)) => Node(<java.lang.Exception: void <init>(java.lang.Throwable)>,l1 := @parameter0: java.lang.Throwable,-1,SimpleNode, path: (<org.jdbi.jdbi3.Handle$1: org.jdbi.jdbi3.ResultSetRow computeNext()>, specialinvoke $stack10.<org.jdbi.jdbi3.JDBIException: void <init>(java.lang.Throwable)>(e), 55) => (<org.jdbi.jdbi3.JDBIException: void <init>(java.lang.Throwable)>, specialinvoke this.<java.lang.RuntimeException: void <init>(java.lang.Throwable)>(cause), 7) => (<java.lang.RuntimeException: void <init>(java.lang.Throwable)>, specialinvoke l0.<java.lang.Exception: void <init>(java.lang.Throwable)>(l1), 96)) => Node(<java.lang.Exception: void <init>(java.lang.Throwable)>,specialinvoke l0.<java.lang.Throwable: void <init>(java.lang.Throwable)>(l1),102,SinkNode, path: (<org.jdbi.jdbi3.Handle$1: org.jdbi.jdbi3.ResultSetRow computeNext()>, specialinvoke $stack10.<org.jdbi.jdbi3.JDBIException: void <init>(java.lang.Throwable)>(e), 55) => (<org.jdbi.jdbi3.JDBIException: void <init>(java.lang.Throwable)>, specialinvoke this.<java.lang.RuntimeException: void <init>(java.lang.Throwable)>(cause), 7) => (<java.lang.RuntimeException: void <init>(java.lang.Throwable)>, specialinvoke l0.<java.lang.Exception: void <init>(java.lang.Throwable)>(l1), 96) => (<java.lang.Exception: void <init>(java.lang.Throwable)>, specialinvoke l0.<java.lang.Throwable: void <init>(java.lang.Throwable)>(l1), 102))
SINK=>BASE: Node(<org.jdbi.jdbi3.Handle$1: org.jdbi.jdbi3.ResultSetRow computeNext()>,$stack9 := @caughtexception,45,SourceNode, path: (<org.jdbi.jdbi3.Handle$1: org.jdbi.jdbi3.ResultSetRow computeNext()>, e = $stack9, 53)) => Node(<org.jdbi.jdbi3.Handle$1: org.jdbi.jdbi3.ResultSetRow computeNext()>,e = $stack9,53,SourceNode, path: (<org.jdbi.jdbi3.Handle$1: org.jdbi.jdbi3.ResultSetRow computeNext()>, e = $stack9, 53)) => Node(<org.jdbi.jdbi3.JDBIException: void <init>(java.lang.Throwable)>,cause := @parameter0: java.lang.Throwable,-1,SimpleNode, path: (<org.jdbi.jdbi3.Handle$1: org.jdbi.jdbi3.ResultSetRow computeNext()>, specialinvoke $stack10.<org.jdbi.jdbi3.JDBIException: void <init>(java.lang.Throwable)>(e), 55)) => Node(<java.lang.RuntimeException: void <init>(java.lang.Throwable)>,l1 := @parameter0: java.lang.Throwable,-1,SimpleNode, path: (<org.jdbi.jdbi3.Handle$1: org.jdbi.jdbi3.ResultSetRow computeNext()>, specialinvoke $stack10.<org.jdbi.jdbi3.JDBIException: void <init>(java.lang.Throwable)>(e), 55) => (<org.jdbi.jdbi3.JDBIException: void <init>(java.lang.Throwable)>, specialinvoke this.<java.lang.RuntimeException: void <init>(java.lang.Throwable)>(cause), 7)) => Node(<java.lang.Exception: void <init>(java.lang.Throwable)>,l1 := @parameter0: java.lang.Throwable,-1,SimpleNode, path: (<org.jdbi.jdbi3.Handle$1: org.jdbi.jdbi3.ResultSetRow computeNext()>, specialinvoke $stack10.<org.jdbi.jdbi3.JDBIException: void <init>(java.lang.Throwable)>(e), 55) => (<org.jdbi.jdbi3.JDBIException: void <init>(java.lang.Throwable)>, specialinvoke this.<java.lang.RuntimeException: void <init>(java.lang.Throwable)>(cause), 7) => (<java.lang.RuntimeException: void <init>(java.lang.Throwable)>, specialinvoke l0.<java.lang.Exception: void <init>(java.lang.Throwable)>(l1), 96)) => Node(<java.lang.Exception: void <init>(java.lang.Throwable)>,specialinvoke l0.<java.lang.Throwable: void <init>(java.lang.Throwable)>(l1),102,SinkNode, path: (<org.jdbi.jdbi3.Handle$1: org.jdbi.jdbi3.ResultSetRow computeNext()>, specialinvoke $stack10.<org.jdbi.jdbi3.JDBIException: void <init>(java.lang.Throwable)>(e), 55) => (<org.jdbi.jdbi3.JDBIException: void <init>(java.lang.Throwable)>, specialinvoke this.<java.lang.RuntimeException: void <init>(java.lang.Throwable)>(cause), 7) => (<java.lang.RuntimeException: void <init>(java.lang.Throwable)>, specialinvoke l0.<java.lang.Exception: void <init>(java.lang.Throwable)>(l1), 96) => (<java.lang.Exception: void <init>(java.lang.Throwable)>, specialinvoke l0.<java.lang.Throwable: void <init>(java.lang.Throwable)>(l1), 102))

SOURCE=>BASE: Node(<org.jdbi.jdbi3.Handle$1: org.jdbi.jdbi3.ResultSetRow computeNext()>,$stack10 = new org.jdbi.jdbi3.JDBIException,55,SourceNode, path: (<org.jdbi.jdbi3.Handle$1: org.jdbi.jdbi3.ResultSetRow computeNext()>, specialinvoke $stack10.<org.jdbi.jdbi3.JDBIException: void <init>(java.lang.Throwable)>(e), 55)) => Node(<org.jdbi.jdbi3.JDBIException: void <init>(java.lang.Throwable)>,this := @this: org.jdbi.jdbi3.JDBIException,-1,SimpleNode, path: (<org.jdbi.jdbi3.Handle$1: org.jdbi.jdbi3.ResultSetRow computeNext()>, specialinvoke $stack10.<org.jdbi.jdbi3.JDBIException: void <init>(java.lang.Throwable)>(e), 55)) => Node(<java.lang.RuntimeException: void <init>(java.lang.Throwable)>,l0 := @this: java.lang.RuntimeException,-1,SimpleNode, path: (<org.jdbi.jdbi3.Handle$1: org.jdbi.jdbi3.ResultSetRow computeNext()>, specialinvoke $stack10.<org.jdbi.jdbi3.JDBIException: void <init>(java.lang.Throwable)>(e), 55) => (<org.jdbi.jdbi3.JDBIException: void <init>(java.lang.Throwable)>, specialinvoke this.<java.lang.RuntimeException: void <init>(java.lang.Throwable)>(cause), 7)) => Node(<java.lang.RuntimeException: void <init>(java.lang.Throwable)>,specialinvoke l0.<java.lang.Exception: void <init>(java.lang.Throwable)>(l1),96,SinkNode, path: (<org.jdbi.jdbi3.Handle$1: org.jdbi.jdbi3.ResultSetRow computeNext()>, specialinvoke $stack10.<org.jdbi.jdbi3.JDBIException: void <init>(java.lang.Throwable)>(e), 55) => (<org.jdbi.jdbi3.JDBIException: void <init>(java.lang.Throwable)>, specialinvoke this.<java.lang.RuntimeException: void <init>(java.lang.Throwable)>(cause), 7) => (<java.lang.RuntimeException: void <init>(java.lang.Throwable)>, specialinvoke l0.<java.lang.Exception: void <init>(java.lang.Throwable)>(l1), 96))
SINK=>BASE: Node(<org.jdbi.jdbi3.Handle$1: org.jdbi.jdbi3.ResultSetRow computeNext()>,$stack9 := @caughtexception,45,SourceNode, path: (<org.jdbi.jdbi3.Handle$1: org.jdbi.jdbi3.ResultSetRow computeNext()>, e = $stack9, 53)) => Node(<org.jdbi.jdbi3.Handle$1: org.jdbi.jdbi3.ResultSetRow computeNext()>,e = $stack9,53,SourceNode, path: (<org.jdbi.jdbi3.Handle$1: org.jdbi.jdbi3.ResultSetRow computeNext()>, e = $stack9, 53)) => Node(<org.jdbi.jdbi3.JDBIException: void <init>(java.lang.Throwable)>,cause := @parameter0: java.lang.Throwable,-1,SimpleNode, path: (<org.jdbi.jdbi3.Handle$1: org.jdbi.jdbi3.ResultSetRow computeNext()>, specialinvoke $stack10.<org.jdbi.jdbi3.JDBIException: void <init>(java.lang.Throwable)>(e), 55)) => Node(<java.lang.RuntimeException: void <init>(java.lang.Throwable)>,l1 := @parameter0: java.lang.Throwable,-1,SimpleNode, path: (<org.jdbi.jdbi3.Handle$1: org.jdbi.jdbi3.ResultSetRow computeNext()>, specialinvoke $stack10.<org.jdbi.jdbi3.JDBIException: void <init>(java.lang.Throwable)>(e), 55) => (<org.jdbi.jdbi3.JDBIException: void <init>(java.lang.Throwable)>, specialinvoke this.<java.lang.RuntimeException: void <init>(java.lang.Throwable)>(cause), 7)) => Node(<java.lang.RuntimeException: void <init>(java.lang.Throwable)>,specialinvoke l0.<java.lang.Exception: void <init>(java.lang.Throwable)>(l1),96,SinkNode, path: (<org.jdbi.jdbi3.Handle$1: org.jdbi.jdbi3.ResultSetRow computeNext()>, specialinvoke $stack10.<org.jdbi.jdbi3.JDBIException: void <init>(java.lang.Throwable)>(e), 55) => (<org.jdbi.jdbi3.JDBIException: void <init>(java.lang.Throwable)>, specialinvoke this.<java.lang.RuntimeException: void <init>(java.lang.Throwable)>(cause), 7) => (<java.lang.RuntimeException: void <init>(java.lang.Throwable)>, specialinvoke l0.<java.lang.Exception: void <init>(java.lang.Throwable)>(l1), 96))

SOURCE=>BASE: Node(<org.jdbi.jdbi3.Handle: java.util.stream.Stream query(java.lang.String)>,$stack11 := @caughtexception,60,SourceNode, path: (<org.jdbi.jdbi3.Handle: java.util.stream.Stream query(java.lang.String)>, stmt#2 = $stack11, 62)) => Node(<org.jdbi.jdbi3.Handle: java.util.stream.Stream query(java.lang.String)>,stmt#2 = $stack11,62,SinkNode, path: (<org.jdbi.jdbi3.Handle: java.util.stream.Stream query(java.lang.String)>, stmt#2 = $stack11, 62)) => Node(<org.jdbi.jdbi3.JDBIException: void <init>(java.lang.Throwable)>,cause := @parameter0: java.lang.Throwable,-1,SimpleNode, path: (<org.jdbi.jdbi3.Handle$1: org.jdbi.jdbi3.ResultSetRow computeNext()>, specialinvoke $stack10.<org.jdbi.jdbi3.JDBIException: void <init>(java.lang.Throwable)>(e), 55)) => Node(<org.jdbi.jdbi3.JDBIException: void <init>(java.lang.Throwable)>,specialinvoke this.<java.lang.RuntimeException: void <init>(java.lang.Throwable)>(cause),7,SinkNode, path: (<org.jdbi.jdbi3.Handle$1: org.jdbi.jdbi3.ResultSetRow computeNext()>, specialinvoke $stack10.<org.jdbi.jdbi3.JDBIException: void <init>(java.lang.Throwable)>(e), 55) => (<org.jdbi.jdbi3.JDBIException: void <init>(java.lang.Throwable)>, specialinvoke this.<java.lang.RuntimeException: void <init>(java.lang.Throwable)>(cause), 7))
SINK=>BASE: Node(<org.jdbi.jdbi3.Handle: java.util.stream.Stream query(java.lang.String)>,$stack11 := @caughtexception,60,SourceNode, path: (<org.jdbi.jdbi3.Handle: java.util.stream.Stream query(java.lang.String)>, stmt#2 = $stack11, 62)) => Node(<org.jdbi.jdbi3.Handle: java.util.stream.Stream query(java.lang.String)>,stmt#2 = $stack11,62,SinkNode, path: (<org.jdbi.jdbi3.Handle: java.util.stream.Stream query(java.lang.String)>, stmt#2 = $stack11, 62)) => Node(<org.jdbi.jdbi3.JDBIException: void <init>(java.lang.Throwable)>,cause := @parameter0: java.lang.Throwable,-1,SimpleNode, path: (<org.jdbi.jdbi3.Handle$1: org.jdbi.jdbi3.ResultSetRow computeNext()>, specialinvoke $stack10.<org.jdbi.jdbi3.JDBIException: void <init>(java.lang.Throwable)>(e), 55)) => Node(<org.jdbi.jdbi3.JDBIException: void <init>(java.lang.Throwable)>,specialinvoke this.<java.lang.RuntimeException: void <init>(java.lang.Throwable)>(cause),7,SinkNode, path: (<org.jdbi.jdbi3.Handle$1: org.jdbi.jdbi3.ResultSetRow computeNext()>, specialinvoke $stack10.<org.jdbi.jdbi3.JDBIException: void <init>(java.lang.Throwable)>(e), 55) => (<org.jdbi.jdbi3.JDBIException: void <init>(java.lang.Throwable)>, specialinvoke this.<java.lang.RuntimeException: void <init>(java.lang.Throwable)>(cause), 7))

SOURCE=>BASE: Node(<org.jdbi.jdbi3.Handle: java.util.stream.Stream query(java.lang.String)>,$stack11 := @caughtexception,60,SourceNode, path: (<org.jdbi.jdbi3.Handle: java.util.stream.Stream query(java.lang.String)>, stmt#2 = $stack11, 62)) => Node(<org.jdbi.jdbi3.Handle: java.util.stream.Stream query(java.lang.String)>,stmt#2 = $stack11,62,SinkNode, path: (<org.jdbi.jdbi3.Handle: java.util.stream.Stream query(java.lang.String)>, stmt#2 = $stack11, 62)) => Node(<org.jdbi.jdbi3.JDBIException: void <init>(java.lang.Throwable)>,cause := @parameter0: java.lang.Throwable,-1,SimpleNode, path: (<org.jdbi.jdbi3.Handle$1: org.jdbi.jdbi3.ResultSetRow computeNext()>, specialinvoke $stack10.<org.jdbi.jdbi3.JDBIException: void <init>(java.lang.Throwable)>(e), 55)) => Node(<java.lang.RuntimeException: void <init>(java.lang.Throwable)>,l1 := @parameter0: java.lang.Throwable,-1,SimpleNode, path: (<org.jdbi.jdbi3.Handle$1: org.jdbi.jdbi3.ResultSetRow computeNext()>, specialinvoke $stack10.<org.jdbi.jdbi3.JDBIException: void <init>(java.lang.Throwable)>(e), 55) => (<org.jdbi.jdbi3.JDBIException: void <init>(java.lang.Throwable)>, specialinvoke this.<java.lang.RuntimeException: void <init>(java.lang.Throwable)>(cause), 7)) => Node(<java.lang.Exception: void <init>(java.lang.Throwable)>,l1 := @parameter0: java.lang.Throwable,-1,SimpleNode, path: (<org.jdbi.jdbi3.Handle$1: org.jdbi.jdbi3.ResultSetRow computeNext()>, specialinvoke $stack10.<org.jdbi.jdbi3.JDBIException: void <init>(java.lang.Throwable)>(e), 55) => (<org.jdbi.jdbi3.JDBIException: void <init>(java.lang.Throwable)>, specialinvoke this.<java.lang.RuntimeException: void <init>(java.lang.Throwable)>(cause), 7) => (<java.lang.RuntimeException: void <init>(java.lang.Throwable)>, specialinvoke l0.<java.lang.Exception: void <init>(java.lang.Throwable)>(l1), 96)) => Node(<java.lang.Exception: void <init>(java.lang.Throwable)>,specialinvoke l0.<java.lang.Throwable: void <init>(java.lang.Throwable)>(l1),102,SinkNode, path: (<org.jdbi.jdbi3.Handle$1: org.jdbi.jdbi3.ResultSetRow computeNext()>, specialinvoke $stack10.<org.jdbi.jdbi3.JDBIException: void <init>(java.lang.Throwable)>(e), 55) => (<org.jdbi.jdbi3.JDBIException: void <init>(java.lang.Throwable)>, specialinvoke this.<java.lang.RuntimeException: void <init>(java.lang.Throwable)>(cause), 7) => (<java.lang.RuntimeException: void <init>(java.lang.Throwable)>, specialinvoke l0.<java.lang.Exception: void <init>(java.lang.Throwable)>(l1), 96) => (<java.lang.Exception: void <init>(java.lang.Throwable)>, specialinvoke l0.<java.lang.Throwable: void <init>(java.lang.Throwable)>(l1), 102))
SINK=>BASE: Node(<org.jdbi.jdbi3.Handle: java.util.stream.Stream query(java.lang.String)>,$stack11 := @caughtexception,60,SourceNode, path: (<org.jdbi.jdbi3.Handle: java.util.stream.Stream query(java.lang.String)>, stmt#2 = $stack11, 62)) => Node(<org.jdbi.jdbi3.Handle: java.util.stream.Stream query(java.lang.String)>,stmt#2 = $stack11,62,SinkNode, path: (<org.jdbi.jdbi3.Handle: java.util.stream.Stream query(java.lang.String)>, stmt#2 = $stack11, 62)) => Node(<org.jdbi.jdbi3.JDBIException: void <init>(java.lang.Throwable)>,cause := @parameter0: java.lang.Throwable,-1,SimpleNode, path: (<org.jdbi.jdbi3.Handle$1: org.jdbi.jdbi3.ResultSetRow computeNext()>, specialinvoke $stack10.<org.jdbi.jdbi3.JDBIException: void <init>(java.lang.Throwable)>(e), 55)) => Node(<java.lang.RuntimeException: void <init>(java.lang.Throwable)>,l1 := @parameter0: java.lang.Throwable,-1,SimpleNode, path: (<org.jdbi.jdbi3.Handle$1: org.jdbi.jdbi3.ResultSetRow computeNext()>, specialinvoke $stack10.<org.jdbi.jdbi3.JDBIException: void <init>(java.lang.Throwable)>(e), 55) => (<org.jdbi.jdbi3.JDBIException: void <init>(java.lang.Throwable)>, specialinvoke this.<java.lang.RuntimeException: void <init>(java.lang.Throwable)>(cause), 7)) => Node(<java.lang.Exception: void <init>(java.lang.Throwable)>,l1 := @parameter0: java.lang.Throwable,-1,SimpleNode, path: (<org.jdbi.jdbi3.Handle$1: org.jdbi.jdbi3.ResultSetRow computeNext()>, specialinvoke $stack10.<org.jdbi.jdbi3.JDBIException: void <init>(java.lang.Throwable)>(e), 55) => (<org.jdbi.jdbi3.JDBIException: void <init>(java.lang.Throwable)>, specialinvoke this.<java.lang.RuntimeException: void <init>(java.lang.Throwable)>(cause), 7) => (<java.lang.RuntimeException: void <init>(java.lang.Throwable)>, specialinvoke l0.<java.lang.Exception: void <init>(java.lang.Throwable)>(l1), 96)) => Node(<java.lang.Exception: void <init>(java.lang.Throwable)>,specialinvoke l0.<java.lang.Throwable: void <init>(java.lang.Throwable)>(l1),102,SinkNode, path: (<org.jdbi.jdbi3.Handle$1: org.jdbi.jdbi3.ResultSetRow computeNext()>, specialinvoke $stack10.<org.jdbi.jdbi3.JDBIException: void <init>(java.lang.Throwable)>(e), 55) => (<org.jdbi.jdbi3.JDBIException: void <init>(java.lang.Throwable)>, specialinvoke this.<java.lang.RuntimeException: void <init>(java.lang.Throwable)>(cause), 7) => (<java.lang.RuntimeException: void <init>(java.lang.Throwable)>, specialinvoke l0.<java.lang.Exception: void <init>(java.lang.Throwable)>(l1), 96) => (<java.lang.Exception: void <init>(java.lang.Throwable)>, specialinvoke l0.<java.lang.Throwable: void <init>(java.lang.Throwable)>(l1), 102))

SOURCE=>BASE: Node(<org.jdbi.jdbi3.Handle: java.util.stream.Stream query(java.lang.String)>,$stack11 := @caughtexception,60,SourceNode, path: (<org.jdbi.jdbi3.Handle: java.util.stream.Stream query(java.lang.String)>, stmt#2 = $stack11, 62)) => Node(<org.jdbi.jdbi3.Handle: java.util.stream.Stream query(java.lang.String)>,stmt#2 = $stack11,62,SinkNode, path: (<org.jdbi.jdbi3.Handle: java.util.stream.Stream query(java.lang.String)>, stmt#2 = $stack11, 62))
SINK=>BASE: Node(<org.jdbi.jdbi3.Handle: java.util.stream.Stream query(java.lang.String)>,$stack11 := @caughtexception,60,SourceNode, path: (<org.jdbi.jdbi3.Handle: java.util.stream.Stream query(java.lang.String)>, stmt#2 = $stack11, 62)) => Node(<org.jdbi.jdbi3.Handle: java.util.stream.Stream query(java.lang.String)>,stmt#2 = $stack11,62,SinkNode, path: (<org.jdbi.jdbi3.Handle: java.util.stream.Stream query(java.lang.String)>, stmt#2 = $stack11, 62))

SOURCE=>BASE: Node(<org.jdbi.jdbi3.Handle: java.util.stream.Stream query(java.lang.String)>,$stack11 := @caughtexception,60,SourceNode, path: (<org.jdbi.jdbi3.Handle: java.util.stream.Stream query(java.lang.String)>, stmt#2 = $stack11, 62)) => Node(<org.jdbi.jdbi3.Handle: java.util.stream.Stream query(java.lang.String)>,stmt#2 = $stack11,62,SinkNode, path: (<org.jdbi.jdbi3.Handle: java.util.stream.Stream query(java.lang.String)>, stmt#2 = $stack11, 62)) => Node(<org.jdbi.jdbi3.JDBIException: void <init>(java.lang.Throwable)>,cause := @parameter0: java.lang.Throwable,-1,SimpleNode, path: (<org.jdbi.jdbi3.Handle$1: org.jdbi.jdbi3.ResultSetRow computeNext()>, specialinvoke $stack10.<org.jdbi.jdbi3.JDBIException: void <init>(java.lang.Throwable)>(e), 55)) => Node(<org.jdbi.jdbi3.JDBIException: void <init>(java.lang.Throwable)>,specialinvoke this.<java.lang.RuntimeException: void <init>(java.lang.Throwable)>(cause),7,SinkNode, path: (<org.jdbi.jdbi3.Handle$1: org.jdbi.jdbi3.ResultSetRow computeNext()>, specialinvoke $stack10.<org.jdbi.jdbi3.JDBIException: void <init>(java.lang.Throwable)>(e), 55) => (<org.jdbi.jdbi3.JDBIException: void <init>(java.lang.Throwable)>, specialinvoke this.<java.lang.RuntimeException: void <init>(java.lang.Throwable)>(cause), 7))
SINK=>BASE: Node(<org.jdbi.jdbi3.Handle$1: org.jdbi.jdbi3.ResultSetRow computeNext()>,e = $stack9,53,SourceNode, path: (<org.jdbi.jdbi3.Handle$1: org.jdbi.jdbi3.ResultSetRow computeNext()>, e = $stack9, 53)) => Node(<org.jdbi.jdbi3.JDBIException: void <init>(java.lang.Throwable)>,cause := @parameter0: java.lang.Throwable,-1,SimpleNode, path: (<org.jdbi.jdbi3.Handle$1: org.jdbi.jdbi3.ResultSetRow computeNext()>, specialinvoke $stack10.<org.jdbi.jdbi3.JDBIException: void <init>(java.lang.Throwable)>(e), 55)) => Node(<org.jdbi.jdbi3.JDBIException: void <init>(java.lang.Throwable)>,specialinvoke this.<java.lang.RuntimeException: void <init>(java.lang.Throwable)>(cause),7,SinkNode, path: (<org.jdbi.jdbi3.Handle$1: org.jdbi.jdbi3.ResultSetRow computeNext()>, specialinvoke $stack10.<org.jdbi.jdbi3.JDBIException: void <init>(java.lang.Throwable)>(e), 55) => (<org.jdbi.jdbi3.JDBIException: void <init>(java.lang.Throwable)>, specialinvoke this.<java.lang.RuntimeException: void <init>(java.lang.Throwable)>(cause), 7))

SOURCE=>BASE: Node(<org.jdbi.jdbi3.Handle$1: org.jdbi.jdbi3.ResultSetRow computeNext()>,$stack9 := @caughtexception,45,SourceNode, path: (<org.jdbi.jdbi3.Handle$1: org.jdbi.jdbi3.ResultSetRow computeNext()>, e = $stack9, 53)) => Node(<org.jdbi.jdbi3.Handle$1: org.jdbi.jdbi3.ResultSetRow computeNext()>,e = $stack9,53,SimpleNode, path: (<org.jdbi.jdbi3.Handle$1: org.jdbi.jdbi3.ResultSetRow computeNext()>, e = $stack9, 53)) => Node(<org.jdbi.jdbi3.JDBIException: void <init>(java.lang.Throwable)>,cause := @parameter0: java.lang.Throwable,-1,SimpleNode, path: (<org.jdbi.jdbi3.Handle$1: org.jdbi.jdbi3.ResultSetRow computeNext()>, specialinvoke $stack10.<org.jdbi.jdbi3.JDBIException: void <init>(java.lang.Throwable)>(e), 55)) => Node(<java.lang.RuntimeException: void <init>(java.lang.Throwable)>,l1 := @parameter0: java.lang.Throwable,-1,SimpleNode, path: (<org.jdbi.jdbi3.Handle$1: org.jdbi.jdbi3.ResultSetRow computeNext()>, specialinvoke $stack10.<org.jdbi.jdbi3.JDBIException: void <init>(java.lang.Throwable)>(e), 55) => (<org.jdbi.jdbi3.JDBIException: void <init>(java.lang.Throwable)>, specialinvoke this.<java.lang.RuntimeException: void <init>(java.lang.Throwable)>(cause), 7)) => Node(<java.lang.Exception: void <init>(java.lang.Throwable)>,l1 := @parameter0: java.lang.Throwable,-1,SimpleNode, path: (<org.jdbi.jdbi3.Handle$1: org.jdbi.jdbi3.ResultSetRow computeNext()>, specialinvoke $stack10.<org.jdbi.jdbi3.JDBIException: void <init>(java.lang.Throwable)>(e), 55) => (<org.jdbi.jdbi3.JDBIException: void <init>(java.lang.Throwable)>, specialinvoke this.<java.lang.RuntimeException: void <init>(java.lang.Throwable)>(cause), 7) => (<java.lang.RuntimeException: void <init>(java.lang.Throwable)>, specialinvoke l0.<java.lang.Exception: void <init>(java.lang.Throwable)>(l1), 96)) => Node(<java.lang.Throwable: void <init>(java.lang.Throwable)>,l1 := @parameter0: java.lang.Throwable,-1,SimpleNode, path: (<org.jdbi.jdbi3.Handle$1: org.jdbi.jdbi3.ResultSetRow computeNext()>, specialinvoke $stack10.<org.jdbi.jdbi3.JDBIException: void <init>(java.lang.Throwable)>(e), 55) => (<org.jdbi.jdbi3.JDBIException: void <init>(java.lang.Throwable)>, specialinvoke this.<java.lang.RuntimeException: void <init>(java.lang.Throwable)>(cause), 7) => (<java.lang.RuntimeException: void <init>(java.lang.Throwable)>, specialinvoke l0.<java.lang.Exception: void <init>(java.lang.Throwable)>(l1), 96) => (<java.lang.Exception: void <init>(java.lang.Throwable)>, specialinvoke l0.<java.lang.Throwable: void <init>(java.lang.Throwable)>(l1), 102)) => Node(<java.lang.Throwable: void <init>(java.lang.Throwable)>,$stack5 = virtualinvoke l1.<java.lang.Throwable: java.lang.String toString()>(),311,SinkNode, path: (<org.jdbi.jdbi3.Handle$1: org.jdbi.jdbi3.ResultSetRow computeNext()>, specialinvoke $stack10.<org.jdbi.jdbi3.JDBIException: void <init>(java.lang.Throwable)>(e), 55) => (<org.jdbi.jdbi3.JDBIException: void <init>(java.lang.Throwable)>, specialinvoke this.<java.lang.RuntimeException: void <init>(java.lang.Throwable)>(cause), 7) => (<java.lang.RuntimeException: void <init>(java.lang.Throwable)>, specialinvoke l0.<java.lang.Exception: void <init>(java.lang.Throwable)>(l1), 96) => (<java.lang.Exception: void <init>(java.lang.Throwable)>, specialinvoke l0.<java.lang.Throwable: void <init>(java.lang.Throwable)>(l1), 102) => (<java.lang.Throwable: void <init>(java.lang.Throwable)>, $stack5 = virtualinvoke l1.<java.lang.Throwable: java.lang.String toString()>(), 311))
SINK=>BASE: Node(<org.jdbi.jdbi3.Handle$1: org.jdbi.jdbi3.ResultSetRow computeNext()>,$stack9 := @caughtexception,45,SourceNode, path: (<org.jdbi.jdbi3.Handle$1: org.jdbi.jdbi3.ResultSetRow computeNext()>, e = $stack9, 53)) => Node(<org.jdbi.jdbi3.Handle$1: org.jdbi.jdbi3.ResultSetRow computeNext()>,e = $stack9,53,SourceNode, path: (<org.jdbi.jdbi3.Handle$1: org.jdbi.jdbi3.ResultSetRow computeNext()>, e = $stack9, 53)) => Node(<org.jdbi.jdbi3.JDBIException: void <init>(java.lang.Throwable)>,cause := @parameter0: java.lang.Throwable,-1,SimpleNode, path: (<org.jdbi.jdbi3.Handle$1: org.jdbi.jdbi3.ResultSetRow computeNext()>, specialinvoke $stack10.<org.jdbi.jdbi3.JDBIException: void <init>(java.lang.Throwable)>(e), 55)) => Node(<java.lang.RuntimeException: void <init>(java.lang.Throwable)>,l1 := @parameter0: java.lang.Throwable,-1,SimpleNode, path: (<org.jdbi.jdbi3.Handle$1: org.jdbi.jdbi3.ResultSetRow computeNext()>, specialinvoke $stack10.<org.jdbi.jdbi3.JDBIException: void <init>(java.lang.Throwable)>(e), 55) => (<org.jdbi.jdbi3.JDBIException: void <init>(java.lang.Throwable)>, specialinvoke this.<java.lang.RuntimeException: void <init>(java.lang.Throwable)>(cause), 7)) => Node(<java.lang.Exception: void <init>(java.lang.Throwable)>,l1 := @parameter0: java.lang.Throwable,-1,SimpleNode, path: (<org.jdbi.jdbi3.Handle$1: org.jdbi.jdbi3.ResultSetRow computeNext()>, specialinvoke $stack10.<org.jdbi.jdbi3.JDBIException: void <init>(java.lang.Throwable)>(e), 55) => (<org.jdbi.jdbi3.JDBIException: void <init>(java.lang.Throwable)>, specialinvoke this.<java.lang.RuntimeException: void <init>(java.lang.Throwable)>(cause), 7) => (<java.lang.RuntimeException: void <init>(java.lang.Throwable)>, specialinvoke l0.<java.lang.Exception: void <init>(java.lang.Throwable)>(l1), 96)) => Node(<java.lang.Throwable: void <init>(java.lang.Throwable)>,l1 := @parameter0: java.lang.Throwable,-1,SimpleNode, path: (<org.jdbi.jdbi3.Handle$1: org.jdbi.jdbi3.ResultSetRow computeNext()>, specialinvoke $stack10.<org.jdbi.jdbi3.JDBIException: void <init>(java.lang.Throwable)>(e), 55) => (<org.jdbi.jdbi3.JDBIException: void <init>(java.lang.Throwable)>, specialinvoke this.<java.lang.RuntimeException: void <init>(java.lang.Throwable)>(cause), 7) => (<java.lang.RuntimeException: void <init>(java.lang.Throwable)>, specialinvoke l0.<java.lang.Exception: void <init>(java.lang.Throwable)>(l1), 96) => (<java.lang.Exception: void <init>(java.lang.Throwable)>, specialinvoke l0.<java.lang.Throwable: void <init>(java.lang.Throwable)>(l1), 102)) => Node(<java.lang.Throwable: void <init>(java.lang.Throwable)>,$stack5 = virtualinvoke l1.<java.lang.Throwable: java.lang.String toString()>(),311,SinkNode, path: (<org.jdbi.jdbi3.Handle$1: org.jdbi.jdbi3.ResultSetRow computeNext()>, specialinvoke $stack10.<org.jdbi.jdbi3.JDBIException: void <init>(java.lang.Throwable)>(e), 55) => (<org.jdbi.jdbi3.JDBIException: void <init>(java.lang.Throwable)>, specialinvoke this.<java.lang.RuntimeException: void <init>(java.lang.Throwable)>(cause), 7) => (<java.lang.RuntimeException: void <init>(java.lang.Throwable)>, specialinvoke l0.<java.lang.Exception: void <init>(java.lang.Throwable)>(l1), 96) => (<java.lang.Exception: void <init>(java.lang.Throwable)>, specialinvoke l0.<java.lang.Throwable: void <init>(java.lang.Throwable)>(l1), 102) => (<java.lang.Throwable: void <init>(java.lang.Throwable)>, $stack5 = virtualinvoke l1.<java.lang.Throwable: java.lang.String toString()>(), 311))

SOURCE=>BASE: Node(<org.jdbi.jdbi3.Handle$1: org.jdbi.jdbi3.ResultSetRow computeNext()>,$stack9 := @caughtexception,45,SourceNode, path: (<org.jdbi.jdbi3.Handle$1: org.jdbi.jdbi3.ResultSetRow computeNext()>, e = $stack9, 53)) => Node(<org.jdbi.jdbi3.Handle$1: org.jdbi.jdbi3.ResultSetRow computeNext()>,e = $stack9,53,SimpleNode, path: (<org.jdbi.jdbi3.Handle$1: org.jdbi.jdbi3.ResultSetRow computeNext()>, e = $stack9, 53)) => Node(<org.jdbi.jdbi3.JDBIException: void <init>(java.lang.Throwable)>,cause := @parameter0: java.lang.Throwable,-1,SimpleNode, path: (<org.jdbi.jdbi3.Handle$1: org.jdbi.jdbi3.ResultSetRow computeNext()>, specialinvoke $stack10.<org.jdbi.jdbi3.JDBIException: void <init>(java.lang.Throwable)>(e), 55)) => Node(<java.lang.RuntimeException: void <init>(java.lang.Throwable)>,l1 := @parameter0: java.lang.Throwable,-1,SimpleNode, path: (<org.jdbi.jdbi3.Handle$1: org.jdbi.jdbi3.ResultSetRow computeNext()>, specialinvoke $stack10.<org.jdbi.jdbi3.JDBIException: void <init>(java.lang.Throwable)>(e), 55) => (<org.jdbi.jdbi3.JDBIException: void <init>(java.lang.Throwable)>, specialinvoke this.<java.lang.RuntimeException: void <init>(java.lang.Throwable)>(cause), 7)) => Node(<java.lang.Exception: void <init>(java.lang.Throwable)>,l1 := @parameter0: java.lang.Throwable,-1,SimpleNode, path: (<org.jdbi.jdbi3.Handle$1: org.jdbi.jdbi3.ResultSetRow computeNext()>, specialinvoke $stack10.<org.jdbi.jdbi3.JDBIException: void <init>(java.lang.Throwable)>(e), 55) => (<org.jdbi.jdbi3.JDBIException: void <init>(java.lang.Throwable)>, specialinvoke this.<java.lang.RuntimeException: void <init>(java.lang.Throwable)>(cause), 7) => (<java.lang.RuntimeException: void <init>(java.lang.Throwable)>, specialinvoke l0.<java.lang.Exception: void <init>(java.lang.Throwable)>(l1), 96)) => Node(<java.lang.Throwable: void <init>(java.lang.Throwable)>,l1 := @parameter0: java.lang.Throwable,-1,SimpleNode, path: (<org.jdbi.jdbi3.Handle$1: org.jdbi.jdbi3.ResultSetRow computeNext()>, specialinvoke $stack10.<org.jdbi.jdbi3.JDBIException: void <init>(java.lang.Throwable)>(e), 55) => (<org.jdbi.jdbi3.JDBIException: void <init>(java.lang.Throwable)>, specialinvoke this.<java.lang.RuntimeException: void <init>(java.lang.Throwable)>(cause), 7) => (<java.lang.RuntimeException: void <init>(java.lang.Throwable)>, specialinvoke l0.<java.lang.Exception: void <init>(java.lang.Throwable)>(l1), 96) => (<java.lang.Exception: void <init>(java.lang.Throwable)>, specialinvoke l0.<java.lang.Throwable: void <init>(java.lang.Throwable)>(l1), 102)) => Node(<java.lang.Throwable: void <init>(java.lang.Throwable)>,$stack5 = virtualinvoke l1.<java.lang.Throwable: java.lang.String toString()>(),311,SinkNode, path: (<org.jdbi.jdbi3.Handle$1: org.jdbi.jdbi3.ResultSetRow computeNext()>, specialinvoke $stack10.<org.jdbi.jdbi3.JDBIException: void <init>(java.lang.Throwable)>(e), 55) => (<org.jdbi.jdbi3.JDBIException: void <init>(java.lang.Throwable)>, specialinvoke this.<java.lang.RuntimeException: void <init>(java.lang.Throwable)>(cause), 7) => (<java.lang.RuntimeException: void <init>(java.lang.Throwable)>, specialinvoke l0.<java.lang.Exception: void <init>(java.lang.Throwable)>(l1), 96) => (<java.lang.Exception: void <init>(java.lang.Throwable)>, specialinvoke l0.<java.lang.Throwable: void <init>(java.lang.Throwable)>(l1), 102) => (<java.lang.Throwable: void <init>(java.lang.Throwable)>, $stack5 = virtualinvoke l1.<java.lang.Throwable: java.lang.String toString()>(), 311))
SINK=>BASE: Node(<org.jdbi.jdbi3.Handle: java.util.stream.Stream query(java.lang.String)>,$stack11 := @caughtexception,60,SourceNode, path: (<org.jdbi.jdbi3.Handle: java.util.stream.Stream query(java.lang.String)>, stmt#2 = $stack11, 62)) => Node(<org.jdbi.jdbi3.Handle: java.util.stream.Stream query(java.lang.String)>,stmt#2 = $stack11,62,SinkNode, path: (<org.jdbi.jdbi3.Handle: java.util.stream.Stream query(java.lang.String)>, stmt#2 = $stack11, 62)) => Node(<org.jdbi.jdbi3.JDBIException: void <init>(java.lang.Throwable)>,cause := @parameter0: java.lang.Throwable,-1,SimpleNode, path: (<org.jdbi.jdbi3.Handle$1: org.jdbi.jdbi3.ResultSetRow computeNext()>, specialinvoke $stack10.<org.jdbi.jdbi3.JDBIException: void <init>(java.lang.Throwable)>(e), 55)) => Node(<java.lang.RuntimeException: void <init>(java.lang.Throwable)>,l1 := @parameter0: java.lang.Throwable,-1,SimpleNode, path: (<org.jdbi.jdbi3.Handle$1: org.jdbi.jdbi3.ResultSetRow computeNext()>, specialinvoke $stack10.<org.jdbi.jdbi3.JDBIException: void <init>(java.lang.Throwable)>(e), 55) => (<org.jdbi.jdbi3.JDBIException: void <init>(java.lang.Throwable)>, specialinvoke this.<java.lang.RuntimeException: void <init>(java.lang.Throwable)>(cause), 7)) => Node(<java.lang.Exception: void <init>(java.lang.Throwable)>,l1 := @parameter0: java.lang.Throwable,-1,SimpleNode, path: (<org.jdbi.jdbi3.Handle$1: org.jdbi.jdbi3.ResultSetRow computeNext()>, specialinvoke $stack10.<org.jdbi.jdbi3.JDBIException: void <init>(java.lang.Throwable)>(e), 55) => (<org.jdbi.jdbi3.JDBIException: void <init>(java.lang.Throwable)>, specialinvoke this.<java.lang.RuntimeException: void <init>(java.lang.Throwable)>(cause), 7) => (<java.lang.RuntimeException: void <init>(java.lang.Throwable)>, specialinvoke l0.<java.lang.Exception: void <init>(java.lang.Throwable)>(l1), 96)) => Node(<java.lang.Throwable: void <init>(java.lang.Throwable)>,l1 := @parameter0: java.lang.Throwable,-1,SimpleNode, path: (<org.jdbi.jdbi3.Handle$1: org.jdbi.jdbi3.ResultSetRow computeNext()>, specialinvoke $stack10.<org.jdbi.jdbi3.JDBIException: void <init>(java.lang.Throwable)>(e), 55) => (<org.jdbi.jdbi3.JDBIException: void <init>(java.lang.Throwable)>, specialinvoke this.<java.lang.RuntimeException: void <init>(java.lang.Throwable)>(cause), 7) => (<java.lang.RuntimeException: void <init>(java.lang.Throwable)>, specialinvoke l0.<java.lang.Exception: void <init>(java.lang.Throwable)>(l1), 96) => (<java.lang.Exception: void <init>(java.lang.Throwable)>, specialinvoke l0.<java.lang.Throwable: void <init>(java.lang.Throwable)>(l1), 102)) => Node(<java.lang.Throwable: void <init>(java.lang.Throwable)>,$stack5 = virtualinvoke l1.<java.lang.Throwable: java.lang.String toString()>(),311,SinkNode, path: (<org.jdbi.jdbi3.Handle$1: org.jdbi.jdbi3.ResultSetRow computeNext()>, specialinvoke $stack10.<org.jdbi.jdbi3.JDBIException: void <init>(java.lang.Throwable)>(e), 55) => (<org.jdbi.jdbi3.JDBIException: void <init>(java.lang.Throwable)>, specialinvoke this.<java.lang.RuntimeException: void <init>(java.lang.Throwable)>(cause), 7) => (<java.lang.RuntimeException: void <init>(java.lang.Throwable)>, specialinvoke l0.<java.lang.Exception: void <init>(java.lang.Throwable)>(l1), 96) => (<java.lang.Exception: void <init>(java.lang.Throwable)>, specialinvoke l0.<java.lang.Throwable: void <init>(java.lang.Throwable)>(l1), 102) => (<java.lang.Throwable: void <init>(java.lang.Throwable)>, $stack5 = virtualinvoke l1.<java.lang.Throwable: java.lang.String toString()>(), 311))

SOURCE=>BASE: Node(<org.jdbi.jdbi3.Handle$1: org.jdbi.jdbi3.ResultSetRow computeNext()>,$stack10 = new org.jdbi.jdbi3.JDBIException,55,SourceNode, path: (<org.jdbi.jdbi3.Handle$1: org.jdbi.jdbi3.ResultSetRow computeNext()>, specialinvoke $stack10.<org.jdbi.jdbi3.JDBIException: void <init>(java.lang.Throwable)>(e), 55)) => Node(<org.jdbi.jdbi3.JDBIException: void <init>(java.lang.Throwable)>,this := @this: org.jdbi.jdbi3.JDBIException,-1,SimpleNode, path: (<org.jdbi.jdbi3.Handle$1: org.jdbi.jdbi3.ResultSetRow computeNext()>, specialinvoke $stack10.<org.jdbi.jdbi3.JDBIException: void <init>(java.lang.Throwable)>(e), 55)) => Node(<java.lang.RuntimeException: void <init>(java.lang.Throwable)>,l0 := @this: java.lang.RuntimeException,-1,SimpleNode, path: (<org.jdbi.jdbi3.Handle$1: org.jdbi.jdbi3.ResultSetRow computeNext()>, specialinvoke $stack10.<org.jdbi.jdbi3.JDBIException: void <init>(java.lang.Throwable)>(e), 55) => (<org.jdbi.jdbi3.JDBIException: void <init>(java.lang.Throwable)>, specialinvoke this.<java.lang.RuntimeException: void <init>(java.lang.Throwable)>(cause), 7)) => Node(<java.lang.Exception: void <init>(java.lang.Throwable)>,l0 := @this: java.lang.Exception,-1,SimpleNode, path: (<org.jdbi.jdbi3.Handle$1: org.jdbi.jdbi3.ResultSetRow computeNext()>, specialinvoke $stack10.<org.jdbi.jdbi3.JDBIException: void <init>(java.lang.Throwable)>(e), 55) => (<org.jdbi.jdbi3.JDBIException: void <init>(java.lang.Throwable)>, specialinvoke this.<java.lang.RuntimeException: void <init>(java.lang.Throwable)>(cause), 7) => (<java.lang.RuntimeException: void <init>(java.lang.Throwable)>, specialinvoke l0.<java.lang.Exception: void <init>(java.lang.Throwable)>(l1), 96)) => Node(<java.lang.Throwable: void <init>(java.lang.Throwable)>,l0 := @this: java.lang.Throwable,-1,SimpleNode, path: (<org.jdbi.jdbi3.Handle$1: org.jdbi.jdbi3.ResultSetRow computeNext()>, specialinvoke $stack10.<org.jdbi.jdbi3.JDBIException: void <init>(java.lang.Throwable)>(e), 55) => (<org.jdbi.jdbi3.JDBIException: void <init>(java.lang.Throwable)>, specialinvoke this.<java.lang.RuntimeException: void <init>(java.lang.Throwable)>(cause), 7) => (<java.lang.RuntimeException: void <init>(java.lang.Throwable)>, specialinvoke l0.<java.lang.Exception: void <init>(java.lang.Throwable)>(l1), 96) => (<java.lang.Exception: void <init>(java.lang.Throwable)>, specialinvoke l0.<java.lang.Throwable: void <init>(java.lang.Throwable)>(l1), 102)) => Node(<java.lang.Throwable: void <init>(java.lang.Throwable)>,l0.<java.lang.Throwable: java.lang.Throwable cause> = l1,312,SinkNode, path: (<org.jdbi.jdbi3.Handle$1: org.jdbi.jdbi3.ResultSetRow computeNext()>, specialinvoke $stack10.<org.jdbi.jdbi3.JDBIException: void <init>(java.lang.Throwable)>(e), 55) => (<org.jdbi.jdbi3.JDBIException: void <init>(java.lang.Throwable)>, specialinvoke this.<java.lang.RuntimeException: void <init>(java.lang.Throwable)>(cause), 7) => (<java.lang.RuntimeException: void <init>(java.lang.Throwable)>, specialinvoke l0.<java.lang.Exception: void <init>(java.lang.Throwable)>(l1), 96) => (<java.lang.Exception: void <init>(java.lang.Throwable)>, specialinvoke l0.<java.lang.Throwable: void <init>(java.lang.Throwable)>(l1), 102) => (<java.lang.Throwable: void <init>(java.lang.Throwable)>, l0.<java.lang.Throwable: java.lang.Throwable cause> = l1, 312))
SINK=>BASE: Node(<org.jdbi.jdbi3.Handle$1: org.jdbi.jdbi3.ResultSetRow computeNext()>,$stack9 := @caughtexception,45,SourceNode, path: (<org.jdbi.jdbi3.Handle$1: org.jdbi.jdbi3.ResultSetRow computeNext()>, e = $stack9, 53)) => Node(<org.jdbi.jdbi3.Handle$1: org.jdbi.jdbi3.ResultSetRow computeNext()>,e = $stack9,53,SourceNode, path: (<org.jdbi.jdbi3.Handle$1: org.jdbi.jdbi3.ResultSetRow computeNext()>, e = $stack9, 53)) => Node(<org.jdbi.jdbi3.JDBIException: void <init>(java.lang.Throwable)>,cause := @parameter0: java.lang.Throwable,-1,SimpleNode, path: (<org.jdbi.jdbi3.Handle$1: org.jdbi.jdbi3.ResultSetRow computeNext()>, specialinvoke $stack10.<org.jdbi.jdbi3.JDBIException: void <init>(java.lang.Throwable)>(e), 55)) => Node(<java.lang.RuntimeException: void <init>(java.lang.Throwable)>,l1 := @parameter0: java.lang.Throwable,-1,SimpleNode, path: (<org.jdbi.jdbi3.Handle$1: org.jdbi.jdbi3.ResultSetRow computeNext()>, specialinvoke $stack10.<org.jdbi.jdbi3.JDBIException: void <init>(java.lang.Throwable)>(e), 55) => (<org.jdbi.jdbi3.JDBIException: void <init>(java.lang.Throwable)>, specialinvoke this.<java.lang.RuntimeException: void <init>(java.lang.Throwable)>(cause), 7)) => Node(<java.lang.Exception: void <init>(java.lang.Throwable)>,l1 := @parameter0: java.lang.Throwable,-1,SimpleNode, path: (<org.jdbi.jdbi3.Handle$1: org.jdbi.jdbi3.ResultSetRow computeNext()>, specialinvoke $stack10.<org.jdbi.jdbi3.JDBIException: void <init>(java.lang.Throwable)>(e), 55) => (<org.jdbi.jdbi3.JDBIException: void <init>(java.lang.Throwable)>, specialinvoke this.<java.lang.RuntimeException: void <init>(java.lang.Throwable)>(cause), 7) => (<java.lang.RuntimeException: void <init>(java.lang.Throwable)>, specialinvoke l0.<java.lang.Exception: void <init>(java.lang.Throwable)>(l1), 96)) => Node(<java.lang.Throwable: void <init>(java.lang.Throwable)>,l1 := @parameter0: java.lang.Throwable,-1,SimpleNode, path: (<org.jdbi.jdbi3.Handle$1: org.jdbi.jdbi3.ResultSetRow computeNext()>, specialinvoke $stack10.<org.jdbi.jdbi3.JDBIException: void <init>(java.lang.Throwable)>(e), 55) => (<org.jdbi.jdbi3.JDBIException: void <init>(java.lang.Throwable)>, specialinvoke this.<java.lang.RuntimeException: void <init>(java.lang.Throwable)>(cause), 7) => (<java.lang.RuntimeException: void <init>(java.lang.Throwable)>, specialinvoke l0.<java.lang.Exception: void <init>(java.lang.Throwable)>(l1), 96) => (<java.lang.Exception: void <init>(java.lang.Throwable)>, specialinvoke l0.<java.lang.Throwable: void <init>(java.lang.Throwable)>(l1), 102)) => Node(<java.lang.Throwable: void <init>(java.lang.Throwable)>,l0.<java.lang.Throwable: java.lang.Throwable cause> = l1,312,SinkNode, path: (<org.jdbi.jdbi3.Handle$1: org.jdbi.jdbi3.ResultSetRow computeNext()>, specialinvoke $stack10.<org.jdbi.jdbi3.JDBIException: void <init>(java.lang.Throwable)>(e), 55) => (<org.jdbi.jdbi3.JDBIException: void <init>(java.lang.Throwable)>, specialinvoke this.<java.lang.RuntimeException: void <init>(java.lang.Throwable)>(cause), 7) => (<java.lang.RuntimeException: void <init>(java.lang.Throwable)>, specialinvoke l0.<java.lang.Exception: void <init>(java.lang.Throwable)>(l1), 96) => (<java.lang.Exception: void <init>(java.lang.Throwable)>, specialinvoke l0.<java.lang.Throwable: void <init>(java.lang.Throwable)>(l1), 102) => (<java.lang.Throwable: void <init>(java.lang.Throwable)>, l0.<java.lang.Throwable: java.lang.Throwable cause> = l1, 312))

SOURCE=>BASE: Node(<org.jdbi.jdbi3.Handle$1: org.jdbi.jdbi3.ResultSetRow computeNext()>,$stack9 := @caughtexception,45,SourceNode, path: (<org.jdbi.jdbi3.Handle$1: org.jdbi.jdbi3.ResultSetRow computeNext()>, e = $stack9, 53)) => Node(<org.jdbi.jdbi3.Handle$1: org.jdbi.jdbi3.ResultSetRow computeNext()>,e = $stack9,53,SimpleNode, path: (<org.jdbi.jdbi3.Handle$1: org.jdbi.jdbi3.ResultSetRow computeNext()>, e = $stack9, 53)) => Node(<org.jdbi.jdbi3.JDBIException: void <init>(java.lang.Throwable)>,cause := @parameter0: java.lang.Throwable,-1,SimpleNode, path: (<org.jdbi.jdbi3.Handle$1: org.jdbi.jdbi3.ResultSetRow computeNext()>, specialinvoke $stack10.<org.jdbi.jdbi3.JDBIException: void <init>(java.lang.Throwable)>(e), 55)) => Node(<java.lang.RuntimeException: void <init>(java.lang.Throwable)>,l1 := @parameter0: java.lang.Throwable,-1,SimpleNode, path: (<org.jdbi.jdbi3.Handle$1: org.jdbi.jdbi3.ResultSetRow computeNext()>, specialinvoke $stack10.<org.jdbi.jdbi3.JDBIException: void <init>(java.lang.Throwable)>(e), 55) => (<org.jdbi.jdbi3.JDBIException: void <init>(java.lang.Throwable)>, specialinvoke this.<java.lang.RuntimeException: void <init>(java.lang.Throwable)>(cause), 7)) => Node(<java.lang.Exception: void <init>(java.lang.Throwable)>,l1 := @parameter0: java.lang.Throwable,-1,SimpleNode, path: (<org.jdbi.jdbi3.Handle$1: org.jdbi.jdbi3.ResultSetRow computeNext()>, specialinvoke $stack10.<org.jdbi.jdbi3.JDBIException: void <init>(java.lang.Throwable)>(e), 55) => (<org.jdbi.jdbi3.JDBIException: void <init>(java.lang.Throwable)>, specialinvoke this.<java.lang.RuntimeException: void <init>(java.lang.Throwable)>(cause), 7) => (<java.lang.RuntimeException: void <init>(java.lang.Throwable)>, specialinvoke l0.<java.lang.Exception: void <init>(java.lang.Throwable)>(l1), 96)) => Node(<java.lang.Throwable: void <init>(java.lang.Throwable)>,l1 := @parameter0: java.lang.Throwable,-1,SimpleNode, path: (<org.jdbi.jdbi3.Handle$1: org.jdbi.jdbi3.ResultSetRow computeNext()>, specialinvoke $stack10.<org.jdbi.jdbi3.JDBIException: void <init>(java.lang.Throwable)>(e), 55) => (<org.jdbi.jdbi3.JDBIException: void <init>(java.lang.Throwable)>, specialinvoke this.<java.lang.RuntimeException: void <init>(java.lang.Throwable)>(cause), 7) => (<java.lang.RuntimeException: void <init>(java.lang.Throwable)>, specialinvoke l0.<java.lang.Exception: void <init>(java.lang.Throwable)>(l1), 96) => (<java.lang.Exception: void <init>(java.lang.Throwable)>, specialinvoke l0.<java.lang.Throwable: void <init>(java.lang.Throwable)>(l1), 102)) => Node(<java.lang.Throwable: void <init>(java.lang.Throwable)>,$stack5 = virtualinvoke l1.<java.lang.Throwable: java.lang.String toString()>(),311,SinkNode, path: (<org.jdbi.jdbi3.Handle$1: org.jdbi.jdbi3.ResultSetRow computeNext()>, specialinvoke $stack10.<org.jdbi.jdbi3.JDBIException: void <init>(java.lang.Throwable)>(e), 55) => (<org.jdbi.jdbi3.JDBIException: void <init>(java.lang.Throwable)>, specialinvoke this.<java.lang.RuntimeException: void <init>(java.lang.Throwable)>(cause), 7) => (<java.lang.RuntimeException: void <init>(java.lang.Throwable)>, specialinvoke l0.<java.lang.Exception: void <init>(java.lang.Throwable)>(l1), 96) => (<java.lang.Exception: void <init>(java.lang.Throwable)>, specialinvoke l0.<java.lang.Throwable: void <init>(java.lang.Throwable)>(l1), 102) => (<java.lang.Throwable: void <init>(java.lang.Throwable)>, $stack5 = virtualinvoke l1.<java.lang.Throwable: java.lang.String toString()>(), 311))
SINK=>BASE: Node(<org.jdbi.jdbi3.Handle$1: org.jdbi.jdbi3.ResultSetRow computeNext()>,e = $stack9,53,SourceNode, path: (<org.jdbi.jdbi3.Handle$1: org.jdbi.jdbi3.ResultSetRow computeNext()>, e = $stack9, 53)) => Node(<org.jdbi.jdbi3.JDBIException: void <init>(java.lang.Throwable)>,cause := @parameter0: java.lang.Throwable,-1,SimpleNode, path: (<org.jdbi.jdbi3.Handle$1: org.jdbi.jdbi3.ResultSetRow computeNext()>, specialinvoke $stack10.<org.jdbi.jdbi3.JDBIException: void <init>(java.lang.Throwable)>(e), 55)) => Node(<java.lang.RuntimeException: void <init>(java.lang.Throwable)>,l1 := @parameter0: java.lang.Throwable,-1,SimpleNode, path: (<org.jdbi.jdbi3.Handle$1: org.jdbi.jdbi3.ResultSetRow computeNext()>, specialinvoke $stack10.<org.jdbi.jdbi3.JDBIException: void <init>(java.lang.Throwable)>(e), 55) => (<org.jdbi.jdbi3.JDBIException: void <init>(java.lang.Throwable)>, specialinvoke this.<java.lang.RuntimeException: void <init>(java.lang.Throwable)>(cause), 7)) => Node(<java.lang.Exception: void <init>(java.lang.Throwable)>,l1 := @parameter0: java.lang.Throwable,-1,SimpleNode, path: (<org.jdbi.jdbi3.Handle$1: org.jdbi.jdbi3.ResultSetRow computeNext()>, specialinvoke $stack10.<org.jdbi.jdbi3.JDBIException: void <init>(java.lang.Throwable)>(e), 55) => (<org.jdbi.jdbi3.JDBIException: void <init>(java.lang.Throwable)>, specialinvoke this.<java.lang.RuntimeException: void <init>(java.lang.Throwable)>(cause), 7) => (<java.lang.RuntimeException: void <init>(java.lang.Throwable)>, specialinvoke l0.<java.lang.Exception: void <init>(java.lang.Throwable)>(l1), 96)) => Node(<java.lang.Throwable: void <init>(java.lang.Throwable)>,l1 := @parameter0: java.lang.Throwable,-1,SimpleNode, path: (<org.jdbi.jdbi3.Handle$1: org.jdbi.jdbi3.ResultSetRow computeNext()>, specialinvoke $stack10.<org.jdbi.jdbi3.JDBIException: void <init>(java.lang.Throwable)>(e), 55) => (<org.jdbi.jdbi3.JDBIException: void <init>(java.lang.Throwable)>, specialinvoke this.<java.lang.RuntimeException: void <init>(java.lang.Throwable)>(cause), 7) => (<java.lang.RuntimeException: void <init>(java.lang.Throwable)>, specialinvoke l0.<java.lang.Exception: void <init>(java.lang.Throwable)>(l1), 96) => (<java.lang.Exception: void <init>(java.lang.Throwable)>, specialinvoke l0.<java.lang.Throwable: void <init>(java.lang.Throwable)>(l1), 102)) => Node(<java.lang.Throwable: void <init>(java.lang.Throwable)>,$stack5 = virtualinvoke l1.<java.lang.Throwable: java.lang.String toString()>(),311,SinkNode, path: (<org.jdbi.jdbi3.Handle$1: org.jdbi.jdbi3.ResultSetRow computeNext()>, specialinvoke $stack10.<org.jdbi.jdbi3.JDBIException: void <init>(java.lang.Throwable)>(e), 55) => (<org.jdbi.jdbi3.JDBIException: void <init>(java.lang.Throwable)>, specialinvoke this.<java.lang.RuntimeException: void <init>(java.lang.Throwable)>(cause), 7) => (<java.lang.RuntimeException: void <init>(java.lang.Throwable)>, specialinvoke l0.<java.lang.Exception: void <init>(java.lang.Throwable)>(l1), 96) => (<java.lang.Exception: void <init>(java.lang.Throwable)>, specialinvoke l0.<java.lang.Throwable: void <init>(java.lang.Throwable)>(l1), 102) => (<java.lang.Throwable: void <init>(java.lang.Throwable)>, $stack5 = virtualinvoke l1.<java.lang.Throwable: java.lang.String toString()>(), 311))

SOURCE=>BASE: Node(<org.jdbi.jdbi3.Handle$1: org.jdbi.jdbi3.ResultSetRow computeNext()>,$stack10 = new org.jdbi.jdbi3.JDBIException,55,SourceNode, path: (<org.jdbi.jdbi3.Handle$1: org.jdbi.jdbi3.ResultSetRow computeNext()>, specialinvoke $stack10.<org.jdbi.jdbi3.JDBIException: void <init>(java.lang.Throwable)>(e), 55)) => Node(<org.jdbi.jdbi3.JDBIException: void <init>(java.lang.Throwable)>,this := @this: org.jdbi.jdbi3.JDBIException,-1,SimpleNode, path: (<org.jdbi.jdbi3.Handle$1: org.jdbi.jdbi3.ResultSetRow computeNext()>, specialinvoke $stack10.<org.jdbi.jdbi3.JDBIException: void <init>(java.lang.Throwable)>(e), 55)) => Node(<java.lang.RuntimeException: void <init>(java.lang.Throwable)>,l0 := @this: java.lang.RuntimeException,-1,SimpleNode, path: (<org.jdbi.jdbi3.Handle$1: org.jdbi.jdbi3.ResultSetRow computeNext()>, specialinvoke $stack10.<org.jdbi.jdbi3.JDBIException: void <init>(java.lang.Throwable)>(e), 55) => (<org.jdbi.jdbi3.JDBIException: void <init>(java.lang.Throwable)>, specialinvoke this.<java.lang.RuntimeException: void <init>(java.lang.Throwable)>(cause), 7)) => Node(<java.lang.Exception: void <init>(java.lang.Throwable)>,l0 := @this: java.lang.Exception,-1,SimpleNode, path: (<org.jdbi.jdbi3.Handle$1: org.jdbi.jdbi3.ResultSetRow computeNext()>, specialinvoke $stack10.<org.jdbi.jdbi3.JDBIException: void <init>(java.lang.Throwable)>(e), 55) => (<org.jdbi.jdbi3.JDBIException: void <init>(java.lang.Throwable)>, specialinvoke this.<java.lang.RuntimeException: void <init>(java.lang.Throwable)>(cause), 7) => (<java.lang.RuntimeException: void <init>(java.lang.Throwable)>, specialinvoke l0.<java.lang.Exception: void <init>(java.lang.Throwable)>(l1), 96)) => Node(<java.lang.Throwable: void <init>(java.lang.Throwable)>,l0 := @this: java.lang.Throwable,-1,SimpleNode, path: (<org.jdbi.jdbi3.Handle$1: org.jdbi.jdbi3.ResultSetRow computeNext()>, specialinvoke $stack10.<org.jdbi.jdbi3.JDBIException: void <init>(java.lang.Throwable)>(e), 55) => (<org.jdbi.jdbi3.JDBIException: void <init>(java.lang.Throwable)>, specialinvoke this.<java.lang.RuntimeException: void <init>(java.lang.Throwable)>(cause), 7) => (<java.lang.RuntimeException: void <init>(java.lang.Throwable)>, specialinvoke l0.<java.lang.Exception: void <init>(java.lang.Throwable)>(l1), 96) => (<java.lang.Exception: void <init>(java.lang.Throwable)>, specialinvoke l0.<java.lang.Throwable: void <init>(java.lang.Throwable)>(l1), 102)) => Node(<java.lang.Throwable: void <init>(java.lang.Throwable)>,l0.<java.lang.Throwable: java.lang.Throwable cause> = l1,312,SinkNode, path: (<org.jdbi.jdbi3.Handle$1: org.jdbi.jdbi3.ResultSetRow computeNext()>, specialinvoke $stack10.<org.jdbi.jdbi3.JDBIException: void <init>(java.lang.Throwable)>(e), 55) => (<org.jdbi.jdbi3.JDBIException: void <init>(java.lang.Throwable)>, specialinvoke this.<java.lang.RuntimeException: void <init>(java.lang.Throwable)>(cause), 7) => (<java.lang.RuntimeException: void <init>(java.lang.Throwable)>, specialinvoke l0.<java.lang.Exception: void <init>(java.lang.Throwable)>(l1), 96) => (<java.lang.Exception: void <init>(java.lang.Throwable)>, specialinvoke l0.<java.lang.Throwable: void <init>(java.lang.Throwable)>(l1), 102) => (<java.lang.Throwable: void <init>(java.lang.Throwable)>, l0.<java.lang.Throwable: java.lang.Throwable cause> = l1, 312))
SINK=>BASE: Node(<org.jdbi.jdbi3.Handle$1: org.jdbi.jdbi3.ResultSetRow computeNext()>,e = $stack9,53,SourceNode, path: (<org.jdbi.jdbi3.Handle$1: org.jdbi.jdbi3.ResultSetRow computeNext()>, e = $stack9, 53)) => Node(<org.jdbi.jdbi3.JDBIException: void <init>(java.lang.Throwable)>,cause := @parameter0: java.lang.Throwable,-1,SimpleNode, path: (<org.jdbi.jdbi3.Handle$1: org.jdbi.jdbi3.ResultSetRow computeNext()>, specialinvoke $stack10.<org.jdbi.jdbi3.JDBIException: void <init>(java.lang.Throwable)>(e), 55)) => Node(<java.lang.RuntimeException: void <init>(java.lang.Throwable)>,l1 := @parameter0: java.lang.Throwable,-1,SimpleNode, path: (<org.jdbi.jdbi3.Handle$1: org.jdbi.jdbi3.ResultSetRow computeNext()>, specialinvoke $stack10.<org.jdbi.jdbi3.JDBIException: void <init>(java.lang.Throwable)>(e), 55) => (<org.jdbi.jdbi3.JDBIException: void <init>(java.lang.Throwable)>, specialinvoke this.<java.lang.RuntimeException: void <init>(java.lang.Throwable)>(cause), 7)) => Node(<java.lang.Exception: void <init>(java.lang.Throwable)>,l1 := @parameter0: java.lang.Throwable,-1,SimpleNode, path: (<org.jdbi.jdbi3.Handle$1: org.jdbi.jdbi3.ResultSetRow computeNext()>, specialinvoke $stack10.<org.jdbi.jdbi3.JDBIException: void <init>(java.lang.Throwable)>(e), 55) => (<org.jdbi.jdbi3.JDBIException: void <init>(java.lang.Throwable)>, specialinvoke this.<java.lang.RuntimeException: void <init>(java.lang.Throwable)>(cause), 7) => (<java.lang.RuntimeException: void <init>(java.lang.Throwable)>, specialinvoke l0.<java.lang.Exception: void <init>(java.lang.Throwable)>(l1), 96)) => Node(<java.lang.Throwable: void <init>(java.lang.Throwable)>,l1 := @parameter0: java.lang.Throwable,-1,SimpleNode, path: (<org.jdbi.jdbi3.Handle$1: org.jdbi.jdbi3.ResultSetRow computeNext()>, specialinvoke $stack10.<org.jdbi.jdbi3.JDBIException: void <init>(java.lang.Throwable)>(e), 55) => (<org.jdbi.jdbi3.JDBIException: void <init>(java.lang.Throwable)>, specialinvoke this.<java.lang.RuntimeException: void <init>(java.lang.Throwable)>(cause), 7) => (<java.lang.RuntimeException: void <init>(java.lang.Throwable)>, specialinvoke l0.<java.lang.Exception: void <init>(java.lang.Throwable)>(l1), 96) => (<java.lang.Exception: void <init>(java.lang.Throwable)>, specialinvoke l0.<java.lang.Throwable: void <init>(java.lang.Throwable)>(l1), 102)) => Node(<java.lang.Throwable: void <init>(java.lang.Throwable)>,l0.<java.lang.Throwable: java.lang.Throwable cause> = l1,312,SinkNode, path: (<org.jdbi.jdbi3.Handle$1: org.jdbi.jdbi3.ResultSetRow computeNext()>, specialinvoke $stack10.<org.jdbi.jdbi3.JDBIException: void <init>(java.lang.Throwable)>(e), 55) => (<org.jdbi.jdbi3.JDBIException: void <init>(java.lang.Throwable)>, specialinvoke this.<java.lang.RuntimeException: void <init>(java.lang.Throwable)>(cause), 7) => (<java.lang.RuntimeException: void <init>(java.lang.Throwable)>, specialinvoke l0.<java.lang.Exception: void <init>(java.lang.Throwable)>(l1), 96) => (<java.lang.Exception: void <init>(java.lang.Throwable)>, specialinvoke l0.<java.lang.Throwable: void <init>(java.lang.Throwable)>(l1), 102) => (<java.lang.Throwable: void <init>(java.lang.Throwable)>, l0.<java.lang.Throwable: java.lang.Throwable cause> = l1, 312))

source(org.jdbi.jdbi3.Handle$1, computeNext, 45, $stack7 = new org.jdbi.jdbi3.ResultSetRow, [at org.jdbi.jdbi3.Handle$1.computeNext(Handle$1.java:45)]) => sink(org.jdbi.jdbi3.Handle$1, computeNext, 45, $stack7 = new org.jdbi.jdbi3.ResultSetRow, [at org.jdbi.jdbi3.Handle$1.computeNext(Handle$1.java:45)])

source(org.jdbi.jdbi3.Handle$1, computeNext, 45, $stack8 = this.<org.jdbi.jdbi3.Handle$1: java.sql.ResultSet val$rs>, [at org.jdbi.jdbi3.Handle$1.computeNext(Handle$1.java:45)]) => sink(org.jdbi.jdbi3.Handle$1, computeNext, 45, $stack8 = this.<org.jdbi.jdbi3.Handle$1: java.sql.ResultSet val$rs>, [at org.jdbi.jdbi3.Handle$1.computeNext(Handle$1.java:45)])

source(org.jdbi.jdbi3.ResultSetRow, <init>, 12, this.<org.jdbi.jdbi3.ResultSetRow: java.sql.ResultSet rs> = rs, [at org.jdbi.jdbi3.Handle$1.computeNext(Handle$1.java:45), at org.jdbi.jdbi3.ResultSetRow.<init>(ResultSetRow.java:12)]) => sink(org.jdbi.jdbi3.ResultSetRow, <init>, 12, this.<org.jdbi.jdbi3.ResultSetRow: java.sql.ResultSet rs> = rs, [at org.jdbi.jdbi3.Handle$1.computeNext(Handle$1.java:45), at org.jdbi.jdbi3.ResultSetRow.<init>(ResultSetRow.java:12)])

List(Node(<org.jdbi.jdbi3.Handle$1: org.jdbi.jdbi3.ResultSetRow computeNext()>,$stack9 := @caughtexception,45,SourceNode, path: (<org.jdbi.jdbi3.Handle$1: org.jdbi.jdbi3.ResultSetRow computeNext()>, e = $stack9, 53)) => Node(<org.jdbi.jdbi3.Handle$1: org.jdbi.jdbi3.ResultSetRow computeNext()>,e = $stack9,53,SinkNode, path: (<org.jdbi.jdbi3.Handle$1: org.jdbi.jdbi3.ResultSetRow computeNext()>, e = $stack9, 53)))

List(Node(<org.jdbi.jdbi3.Handle$1: org.jdbi.jdbi3.ResultSetRow computeNext()>,$stack7 = new org.jdbi.jdbi3.ResultSetRow,45,SourceNode, path: (<org.jdbi.jdbi3.Handle$1: org.jdbi.jdbi3.ResultSetRow computeNext()>, specialinvoke $stack7.<org.jdbi.jdbi3.ResultSetRow: void <init>(java.sql.ResultSet)>($stack8), 45)) => Node(<org.jdbi.jdbi3.ResultSetRow: void <init>(java.sql.ResultSet)>,this := @this: org.jdbi.jdbi3.ResultSetRow,-1,SimpleNode, path: (<org.jdbi.jdbi3.Handle$1: org.jdbi.jdbi3.ResultSetRow computeNext()>, specialinvoke $stack7.<org.jdbi.jdbi3.ResultSetRow: void <init>(java.sql.ResultSet)>($stack8), 45)) => Node(<org.jdbi.jdbi3.ResultSetRow: void <init>(java.sql.ResultSet)>,specialinvoke this.<java.lang.Object: void <init>()>(),10,SinkNode, path: (<org.jdbi.jdbi3.Handle$1: org.jdbi.jdbi3.ResultSetRow computeNext()>, specialinvoke $stack7.<org.jdbi.jdbi3.ResultSetRow: void <init>(java.sql.ResultSet)>($stack8), 45) => (<org.jdbi.jdbi3.ResultSetRow: void <init>(java.sql.ResultSet)>, specialinvoke this.<java.lang.Object: void <init>()>(), 10)))

List(Node(<org.jdbi.jdbi3.Handle$1: org.jdbi.jdbi3.ResultSetRow computeNext()>,$stack7 = new org.jdbi.jdbi3.ResultSetRow,45,SourceNode, path: (<org.jdbi.jdbi3.Handle$1: org.jdbi.jdbi3.ResultSetRow computeNext()>, specialinvoke $stack7.<org.jdbi.jdbi3.ResultSetRow: void <init>(java.sql.ResultSet)>($stack8), 45)) => Node(<org.jdbi.jdbi3.ResultSetRow: void <init>(java.sql.ResultSet)>,this := @this: org.jdbi.jdbi3.ResultSetRow,-1,SimpleNode, path: (<org.jdbi.jdbi3.Handle$1: org.jdbi.jdbi3.ResultSetRow computeNext()>, specialinvoke $stack7.<org.jdbi.jdbi3.ResultSetRow: void <init>(java.sql.ResultSet)>($stack8), 45)) => Node(<org.jdbi.jdbi3.ResultSetRow: void <init>(java.sql.ResultSet)>,specialinvoke this.<java.lang.Object: void <init>()>(),10,SinkNode, path: (<org.jdbi.jdbi3.Handle$1: org.jdbi.jdbi3.ResultSetRow computeNext()>, specialinvoke $stack7.<org.jdbi.jdbi3.ResultSetRow: void <init>(java.sql.ResultSet)>($stack8), 45) => (<org.jdbi.jdbi3.ResultSetRow: void <init>(java.sql.ResultSet)>, specialinvoke this.<java.lang.Object: void <init>()>(), 10)))

List(Node(<org.jdbi.jdbi3.Handle$1: org.jdbi.jdbi3.ResultSetRow computeNext()>,e = $stack9,53,SourceNode, path: (<org.jdbi.jdbi3.Handle$1: org.jdbi.jdbi3.ResultSetRow computeNext()>, e = $stack9, 53)) => Node(<org.jdbi.jdbi3.JDBIException: void <init>(java.lang.Throwable)>,cause := @parameter0: java.lang.Throwable,-1,SimpleNode, path: (<org.jdbi.jdbi3.Handle$1: org.jdbi.jdbi3.ResultSetRow computeNext()>, specialinvoke $stack10.<org.jdbi.jdbi3.JDBIException: void <init>(java.lang.Throwable)>(e), 55)) => Node(<org.jdbi.jdbi3.JDBIException: void <init>(java.lang.Throwable)>,specialinvoke this.<java.lang.RuntimeException: void <init>(java.lang.Throwable)>(cause),7,SinkNode, path: (<org.jdbi.jdbi3.Handle$1: org.jdbi.jdbi3.ResultSetRow computeNext()>, specialinvoke $stack10.<org.jdbi.jdbi3.JDBIException: void <init>(java.lang.Throwable)>(e), 55) => (<org.jdbi.jdbi3.JDBIException: void <init>(java.lang.Throwable)>, specialinvoke this.<java.lang.RuntimeException: void <init>(java.lang.Throwable)>(cause), 7)))

List(Node(<org.jdbi.jdbi3.Handle$1: org.jdbi.jdbi3.ResultSetRow computeNext()>,$stack9 := @caughtexception,45,SourceNode, path: (<org.jdbi.jdbi3.Handle$1: org.jdbi.jdbi3.ResultSetRow computeNext()>, e = $stack9, 53)) => Node(<org.jdbi.jdbi3.Handle$1: org.jdbi.jdbi3.ResultSetRow computeNext()>,e = $stack9,53,SourceNode, path: (<org.jdbi.jdbi3.Handle$1: org.jdbi.jdbi3.ResultSetRow computeNext()>, e = $stack9, 53)) => Node(<org.jdbi.jdbi3.Handle$1: org.jdbi.jdbi3.ResultSetRow computeNext()>,specialinvoke $stack10.<org.jdbi.jdbi3.JDBIException: void <init>(java.lang.Throwable)>(e),55,SinkNode, path: (<org.jdbi.jdbi3.Handle$1: org.jdbi.jdbi3.ResultSetRow computeNext()>, specialinvoke $stack10.<org.jdbi.jdbi3.JDBIException: void <init>(java.lang.Throwable)>(e), 55)))

List(Node(<org.jdbi.jdbi3.Handle: java.util.stream.Stream query(java.lang.String)>,rs = interfaceinvoke stmt.<java.sql.PreparedStatement: java.sql.ResultSet executeQuery()>(),35,SourceNode, path: (<org.jdbi.jdbi3.Handle: java.util.stream.Stream query(java.lang.String)>, rs = interfaceinvoke stmt.<java.sql.PreparedStatement: java.sql.ResultSet executeQuery()>(), 35)) => Node(<org.jdbi.jdbi3.Handle: java.util.stream.Stream query(java.lang.String)>,specialinvoke tmp$79620878.<org.jdbi.jdbi3.Handle$1: void <init>(org.jdbi.jdbi3.Handle,java.sql.ResultSet)>(this, rs),37,SinkNode, path: (<org.jdbi.jdbi3.Handle: java.util.stream.Stream query(java.lang.String)>, specialinvoke tmp$79620878.<org.jdbi.jdbi3.Handle$1: void <init>(org.jdbi.jdbi3.Handle,java.sql.ResultSet)>(this, rs), 37)))

List(Node(<org.jdbi.jdbi3.Handle$1: org.jdbi.jdbi3.ResultSetRow computeNext()>,$stack9 := @caughtexception,45,SourceNode, path: (<org.jdbi.jdbi3.Handle$1: org.jdbi.jdbi3.ResultSetRow computeNext()>, e = $stack9, 53)) => Node(<org.jdbi.jdbi3.Handle$1: org.jdbi.jdbi3.ResultSetRow computeNext()>,e = $stack9,53,SourceNode, path: (<org.jdbi.jdbi3.Handle$1: org.jdbi.jdbi3.ResultSetRow computeNext()>, e = $stack9, 53)) => Node(<org.jdbi.jdbi3.JDBIException: void <init>(java.lang.Throwable)>,cause := @parameter0: java.lang.Throwable,-1,SimpleNode, path: (<org.jdbi.jdbi3.Handle$1: org.jdbi.jdbi3.ResultSetRow computeNext()>, specialinvoke $stack10.<org.jdbi.jdbi3.JDBIException: void <init>(java.lang.Throwable)>(e), 55)) => Node(<org.jdbi.jdbi3.JDBIException: void <init>(java.lang.Throwable)>,specialinvoke this.<java.lang.RuntimeException: void <init>(java.lang.Throwable)>(cause),7,SinkNode, path: (<org.jdbi.jdbi3.Handle$1: org.jdbi.jdbi3.ResultSetRow computeNext()>, specialinvoke $stack10.<org.jdbi.jdbi3.JDBIException: void <init>(java.lang.Throwable)>(e), 55) => (<org.jdbi.jdbi3.JDBIException: void <init>(java.lang.Throwable)>, specialinvoke this.<java.lang.RuntimeException: void <init>(java.lang.Throwable)>(cause), 7)))

List(Node(<org.jdbi.jdbi3.Handle: java.util.stream.Stream query(java.lang.String)>,$stack11 := @caughtexception,60,SourceNode, path: (<org.jdbi.jdbi3.Handle: java.util.stream.Stream query(java.lang.String)>, stmt#2 = $stack11, 62)) => Node(<org.jdbi.jdbi3.Handle: java.util.stream.Stream query(java.lang.String)>,stmt#2 = $stack11,62,SimpleNode, path: (<org.jdbi.jdbi3.Handle: java.util.stream.Stream query(java.lang.String)>, stmt#2 = $stack11, 62)) => Node(<org.jdbi.jdbi3.JDBIException: void <init>(java.lang.Throwable)>,cause := @parameter0: java.lang.Throwable,-1,SimpleNode, path: (<org.jdbi.jdbi3.Handle$1: org.jdbi.jdbi3.ResultSetRow computeNext()>, specialinvoke $stack10.<org.jdbi.jdbi3.JDBIException: void <init>(java.lang.Throwable)>(e), 55)) => Node(<org.jdbi.jdbi3.JDBIException: void <init>(java.lang.Throwable)>,specialinvoke this.<java.lang.RuntimeException: void <init>(java.lang.Throwable)>(cause),7,SinkNode, path: (<org.jdbi.jdbi3.Handle$1: org.jdbi.jdbi3.ResultSetRow computeNext()>, specialinvoke $stack10.<org.jdbi.jdbi3.JDBIException: void <init>(java.lang.Throwable)>(e), 55) => (<org.jdbi.jdbi3.JDBIException: void <init>(java.lang.Throwable)>, specialinvoke this.<java.lang.RuntimeException: void <init>(java.lang.Throwable)>(cause), 7)))

List(Node(<org.jdbi.jdbi3.Handle: java.util.stream.Stream query(java.lang.String)>,rs = interfaceinvoke stmt.<java.sql.PreparedStatement: java.sql.ResultSet executeQuery()>(),35,SourceNode, path: (<org.jdbi.jdbi3.Handle: java.util.stream.Stream query(java.lang.String)>, rs = interfaceinvoke stmt.<java.sql.PreparedStatement: java.sql.ResultSet executeQuery()>(), 35)) => Node(<org.jdbi.jdbi3.Handle$1: void <init>(org.jdbi.jdbi3.Handle,java.sql.ResultSet)>,l2 := @parameter1: java.sql.ResultSet,-1,SimpleNode, path: (<org.jdbi.jdbi3.Handle: java.util.stream.Stream query(java.lang.String)>, specialinvoke tmp$79620878.<org.jdbi.jdbi3.Handle$1: void <init>(org.jdbi.jdbi3.Handle,java.sql.ResultSet)>(this, rs), 37)) => Node(<org.jdbi.jdbi3.Handle$1: void <init>(org.jdbi.jdbi3.Handle,java.sql.ResultSet)>,this.<org.jdbi.jdbi3.Handle$1: java.sql.ResultSet val$rs> = l2,38,SinkNode, path: (<org.jdbi.jdbi3.Handle: java.util.stream.Stream query(java.lang.String)>, specialinvoke tmp$79620878.<org.jdbi.jdbi3.Handle$1: void <init>(org.jdbi.jdbi3.Handle,java.sql.ResultSet)>(this, rs), 37) => (<org.jdbi.jdbi3.Handle$1: void <init>(org.jdbi.jdbi3.Handle,java.sql.ResultSet)>, this.<org.jdbi.jdbi3.Handle$1: java.sql.ResultSet val$rs> = l2, 38)))

List(Node(<org.jdbi.jdbi3.Handle$1: org.jdbi.jdbi3.ResultSetRow computeNext()>,e = $stack9,53,SourceNode, path: (<org.jdbi.jdbi3.Handle$1: org.jdbi.jdbi3.ResultSetRow computeNext()>, e = $stack9, 53)) => Node(<org.jdbi.jdbi3.Handle$1: org.jdbi.jdbi3.ResultSetRow computeNext()>,specialinvoke $stack10.<org.jdbi.jdbi3.JDBIException: void <init>(java.lang.Throwable)>(e),55,SinkNode, path: (<org.jdbi.jdbi3.Handle$1: org.jdbi.jdbi3.ResultSetRow computeNext()>, specialinvoke $stack10.<org.jdbi.jdbi3.JDBIException: void <init>(java.lang.Throwable)>(e), 55)))

List(Node(<org.jdbi.jdbi3.Handle$1: org.jdbi.jdbi3.ResultSetRow computeNext()>,$stack9 := @caughtexception,45,SourceNode, path: (<org.jdbi.jdbi3.Handle$1: org.jdbi.jdbi3.ResultSetRow computeNext()>, e = $stack9, 53)) => Node(<org.jdbi.jdbi3.Handle$1: org.jdbi.jdbi3.ResultSetRow computeNext()>,e = $stack9,53,SinkNode, path: (<org.jdbi.jdbi3.Handle$1: org.jdbi.jdbi3.ResultSetRow computeNext()>, e = $stack9, 53)))

List(Node(<org.jdbi.jdbi3.Handle$1: org.jdbi.jdbi3.ResultSetRow computeNext()>,e = $stack9,53,SourceNode, path: (<org.jdbi.jdbi3.Handle$1: org.jdbi.jdbi3.ResultSetRow computeNext()>, e = $stack9, 53)) => Node(<org.jdbi.jdbi3.Handle$1: org.jdbi.jdbi3.ResultSetRow computeNext()>,specialinvoke $stack10.<org.jdbi.jdbi3.JDBIException: void <init>(java.lang.Throwable)>(e),55,SinkNode, path: (<org.jdbi.jdbi3.Handle$1: org.jdbi.jdbi3.ResultSetRow computeNext()>, specialinvoke $stack10.<org.jdbi.jdbi3.JDBIException: void <init>(java.lang.Throwable)>(e), 55)))

List(Node(<org.jdbi.jdbi3.Handle$1: org.jdbi.jdbi3.ResultSetRow computeNext()>,$stack9 := @caughtexception,45,SourceNode, path: (<org.jdbi.jdbi3.Handle$1: org.jdbi.jdbi3.ResultSetRow computeNext()>, e = $stack9, 53)) => Node(<org.jdbi.jdbi3.Handle$1: org.jdbi.jdbi3.ResultSetRow computeNext()>,e = $stack9,53,SourceNode, path: (<org.jdbi.jdbi3.Handle$1: org.jdbi.jdbi3.ResultSetRow computeNext()>, e = $stack9, 53)) => Node(<org.jdbi.jdbi3.Handle$1: org.jdbi.jdbi3.ResultSetRow computeNext()>,specialinvoke $stack10.<org.jdbi.jdbi3.JDBIException: void <init>(java.lang.Throwable)>(e),55,SinkNode, path: (<org.jdbi.jdbi3.Handle$1: org.jdbi.jdbi3.ResultSetRow computeNext()>, specialinvoke $stack10.<org.jdbi.jdbi3.JDBIException: void <init>(java.lang.Throwable)>(e), 55)))

List(Node(<org.jdbi.jdbi3.Handle: java.util.stream.Stream query(java.lang.String)>,rs = interfaceinvoke stmt.<java.sql.PreparedStatement: java.sql.ResultSet executeQuery()>(),35,SourceNode, path: (<org.jdbi.jdbi3.Handle: java.util.stream.Stream query(java.lang.String)>, rs = interfaceinvoke stmt.<java.sql.PreparedStatement: java.sql.ResultSet executeQuery()>(), 35)) => Node(<org.jdbi.jdbi3.Handle: java.util.stream.Stream query(java.lang.String)>,specialinvoke tmp$1810349933.<org.jdbi.jdbi3.Handle$1: void <init>(org.jdbi.jdbi3.Handle,java.sql.ResultSet)>(this, rs),37,SinkNode, path: (<org.jdbi.jdbi3.Handle: java.util.stream.Stream query(java.lang.String)>, specialinvoke tmp$1810349933.<org.jdbi.jdbi3.Handle$1: void <init>(org.jdbi.jdbi3.Handle,java.sql.ResultSet)>(this, rs), 37)))

